<!DOCTYPE html>
<html lang="en">
    <head>
        <script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"beacon-5.newrelic.com","errorBeacon":"bam.nr-data.net","licenseKey":"598a124f17","applicationID":"3007887","transactionName":"MQcDMkECCkNSW0YMWghNJQlHLQFEcFdcEUcJDg0DQUwlQkdRUQlQSSQUCl83AUhHekshei8=","queueTime":0,"applicationTime":44,"ttGuid":"D579489C4A6F5E01","agent":"js-agent.newrelic.com/nr-476.min.js"}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"VgUHUl5WGwAAVFZaDwY="};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o?o:e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({QJf3ax:[function(t,e){function n(t){function e(e,n,a){t&&t(e,n,a),a||(a={});for(var c=s(e),f=c.length,u=i(a,o,r),d=0;f>d;d++)c[d].apply(u,n);return u}function a(t,e){f[t]=s(t).concat(e)}function s(t){return f[t]||[]}function c(){return n(e)}var f={};return{on:a,emit:e,create:c,listeners:s,_events:f}}function r(){return{}}var o="nr@context",i=t("gos");e.exports=n()},{gos:"7eSDFh"}],ee:[function(t,e){e.exports=t("QJf3ax")},{}],3:[function(t){function e(t,e,n,i,s){try{c?c-=1:r("err",[s||new UncaughtException(t,e,n)])}catch(f){try{r("ierr",[f,(new Date).getTime(),!0])}catch(u){}}return"function"==typeof a?a.apply(this,o(arguments)):!1}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function n(t){r("err",[t,(new Date).getTime()])}var r=t("handle"),o=t(5),i=t("ee"),a=window.onerror,s=!1,c=0;t("loader").features.err=!0,window.onerror=e,NREUM.noticeError=n;try{throw new Error}catch(f){"stack"in f&&(t(1),t(4),"addEventListener"in window&&t(2),window.XMLHttpRequest&&XMLHttpRequest.prototype&&XMLHttpRequest.prototype.addEventListener&&t(3),s=!0)}i.on("fn-start",function(){s&&(c+=1)}),i.on("fn-err",function(t,e,r){s&&(this.thrown=!0,n(r))}),i.on("fn-end",function(){s&&!this.thrown&&c>0&&(c-=1)}),i.on("internal-error",function(t){r("ierr",[t,(new Date).getTime(),!0])})},{1:8,2:5,3:9,4:7,5:20,ee:"QJf3ax",handle:"D5DuLP",loader:"G9z0Bl"}],4:[function(t){function e(){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var n=t("ee"),r=t("handle"),o=t(2);t("loader").features.stn=!0,t(1),n.on("fn-start",function(t){var e=t[0];e instanceof Event&&(this.bstStart=Date.now())}),n.on("fn-end",function(t,e){var n=t[0];n instanceof Event&&r("bst",[n,e,this.bstStart,Date.now()])}),o.on("fn-start",function(t,e,n){this.bstStart=Date.now(),this.bstType=n}),o.on("fn-end",function(t,e){r("bstTimer",[e,this.bstStart,Date.now(),this.bstType])}),n.on("pushState-start",function(){this.time=Date.now(),this.startPath=location.pathname+location.hash}),n.on("pushState-end",function(){r("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),"addEventListener"in window.performance&&(window.performance.addEventListener("webkitresourcetimingbufferfull",function(){r("bstResource",[window.performance.getEntriesByType("resource")]),window.performance.webkitClearResourceTimings()},!1),window.performance.addEventListener("resourcetimingbufferfull",function(){r("bstResource",[window.performance.getEntriesByType("resource")]),window.performance.clearResourceTimings()},!1)),document.addEventListener("scroll",e,!1),document.addEventListener("keypress",e,!1),document.addEventListener("click",e,!1)}},{1:6,2:8,ee:"QJf3ax",handle:"D5DuLP",loader:"G9z0Bl"}],5:[function(t,e){function n(t){i.inPlace(t,["addEventListener","removeEventListener"],"-",r)}function r(t){return t[1]}var o=(t(1),t("ee").create()),i=t(2)(o),a=t("gos");if(e.exports=o,n(window),"getPrototypeOf"in Object){for(var s=document;s&&!s.hasOwnProperty("addEventListener");)s=Object.getPrototypeOf(s);s&&n(s);for(var c=XMLHttpRequest.prototype;c&&!c.hasOwnProperty("addEventListener");)c=Object.getPrototypeOf(c);c&&n(c)}else XMLHttpRequest.prototype.hasOwnProperty("addEventListener")&&n(XMLHttpRequest.prototype);o.on("addEventListener-start",function(t){if(t[1]){var e=t[1];"function"==typeof e?this.wrapped=t[1]=a(e,"nr@wrapped",function(){return i(e,"fn-",null,e.name||"anonymous")}):"function"==typeof e.handleEvent&&i.inPlace(e,["handleEvent"],"fn-")}}),o.on("removeEventListener-start",function(t){var e=this.wrapped;e&&(t[1]=e)})},{1:20,2:21,ee:"QJf3ax",gos:"7eSDFh"}],6:[function(t,e){var n=(t(2),t("ee").create()),r=t(1)(n);e.exports=n,r.inPlace(window.history,["pushState"],"-")},{1:21,2:20,ee:"QJf3ax"}],7:[function(t,e){var n=(t(2),t("ee").create()),r=t(1)(n);e.exports=n,r.inPlace(window,["requestAnimationFrame","mozRequestAnimationFrame","webkitRequestAnimationFrame","msRequestAnimationFrame"],"raf-"),n.on("raf-start",function(t){t[0]=r(t[0],"fn-")})},{1:21,2:20,ee:"QJf3ax"}],8:[function(t,e){function n(t,e,n){var r=t[0];"string"==typeof r&&(r=new Function(r)),t[0]=o(r,"fn-",null,n)}var r=(t(2),t("ee").create()),o=t(1)(r);e.exports=r,o.inPlace(window,["setTimeout","setInterval","setImmediate"],"setTimer-"),r.on("setTimer-start",n)},{1:21,2:20,ee:"QJf3ax"}],9:[function(t,e){function n(){c.inPlace(this,d,"fn-")}function r(t,e){c.inPlace(e,["onreadystatechange"],"fn-")}function o(t,e){return e}var i=t("ee").create(),a=t(1),s=t(2),c=s(i),f=s(a),u=window.XMLHttpRequest,d=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"];e.exports=i,window.XMLHttpRequest=function(t){var e=new u(t);try{i.emit("new-xhr",[],e),f.inPlace(e,["addEventListener","removeEventListener"],"-",function(t,e){return e}),e.addEventListener("readystatechange",n,!1)}catch(r){try{i.emit("internal-error",[r])}catch(o){}}return e},window.XMLHttpRequest.prototype=u.prototype,c.inPlace(XMLHttpRequest.prototype,["open","send"],"-xhr-",o),i.on("send-xhr-start",r),i.on("open-xhr-start",r)},{1:5,2:21,ee:"QJf3ax"}],10:[function(t){function e(t){if("string"==typeof t&&t.length)return t.length;if("object"!=typeof t)return void 0;if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if("undefined"!=typeof FormData&&t instanceof FormData)return void 0;try{return JSON.stringify(t).length}catch(e){return void 0}}function n(t){var n=this.params,r=this.metrics;if(!this.ended){this.ended=!0;for(var i=0;c>i;i++)t.removeEventListener(s[i],this.listener,!1);if(!n.aborted){if(r.duration=(new Date).getTime()-this.startTime,4===t.readyState){n.status=t.status;var a=t.responseType,f="arraybuffer"===a||"blob"===a||"json"===a?t.response:t.responseText,u=e(f);if(u&&(r.rxSize=u),this.sameOrigin){var d=t.getResponseHeader("X-NewRelic-App-Data");d&&(n.cat=d.split(", ").pop())}}else n.status=0;r.cbTime=this.cbTime,o("xhr",[n,r,this.startTime])}}}function r(t,e){var n=i(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}if(window.XMLHttpRequest&&XMLHttpRequest.prototype&&XMLHttpRequest.prototype.addEventListener&&!/CriOS/.test(navigator.userAgent)){t("loader").features.xhr=!0;var o=t("handle"),i=t(2),a=t("ee"),s=["load","error","abort","timeout"],c=s.length,f=t(1);t(4),t(3),a.on("new-xhr",function(){this.totalCbs=0,this.called=0,this.cbTime=0,this.end=n,this.ended=!1,this.xhrGuids={}}),a.on("open-xhr-start",function(t){this.params={method:t[0]},r(this,t[1]),this.metrics={}}),a.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),a.on("send-xhr-start",function(t,n){var r=this.metrics,o=t[0],i=this;if(r&&o){var f=e(o);f&&(r.txSize=f)}this.startTime=(new Date).getTime(),this.listener=function(t){try{"abort"===t.type&&(i.params.aborted=!0),("load"!==t.type||i.called===i.totalCbs&&(i.onloadCalled||"function"!=typeof n.onload))&&i.end(n)}catch(e){try{a.emit("internal-error",[e])}catch(r){}}};for(var u=0;c>u;u++)n.addEventListener(s[u],this.listener,!1)}),a.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),a.on("xhr-load-added",function(t,e){var n=""+f(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),a.on("xhr-load-removed",function(t,e){var n=""+f(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),a.on("addEventListener-end",function(t,e){e instanceof XMLHttpRequest&&"load"===t[0]&&a.emit("xhr-load-added",[t[1],t[2]],e)}),a.on("removeEventListener-end",function(t,e){e instanceof XMLHttpRequest&&"load"===t[0]&&a.emit("xhr-load-removed",[t[1],t[2]],e)}),a.on("fn-start",function(t,e,n){e instanceof XMLHttpRequest&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=(new Date).getTime()))}),a.on("fn-end",function(t,e){this.xhrCbStart&&a.emit("xhr-cb-time",[(new Date).getTime()-this.xhrCbStart,this.onload,e],e)})}},{1:"XL7HBI",2:11,3:9,4:5,ee:"QJf3ax",handle:"D5DuLP",loader:"G9z0Bl"}],11:[function(t,e){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");return!r.port&&o[1]&&(r.port=o[1].split("/")[0].split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname),r.sameOrigin=!e.hostname||e.hostname===document.domain&&e.port===n.port&&e.protocol===n.protocol,r}},{}],gos:[function(t,e){e.exports=t("7eSDFh")},{}],"7eSDFh":[function(t,e){function n(t,e,n){if(r.call(t,e))return t[e];var o=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:o,writable:!0,enumerable:!1}),o}catch(i){}return t[e]=o,o}var r=Object.prototype.hasOwnProperty;e.exports=n},{}],D5DuLP:[function(t,e){function n(t,e,n){return r.listeners(t).length?r.emit(t,e,n):(o[t]||(o[t]=[]),void o[t].push(e))}var r=t("ee").create(),o={};e.exports=n,n.ee=r,r.q=o},{ee:"QJf3ax"}],handle:[function(t,e){e.exports=t("D5DuLP")},{}],XL7HBI:[function(t,e){function n(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:i(t,o,function(){return r++})}var r=1,o="nr@id",i=t("gos");e.exports=n},{gos:"7eSDFh"}],id:[function(t,e){e.exports=t("XL7HBI")},{}],loader:[function(t,e){e.exports=t("G9z0Bl")},{}],G9z0Bl:[function(t,e){function n(){var t=p.info=NREUM.info;if(t&&t.agent&&t.licenseKey&&t.applicationID&&c&&c.body){p.proto="https"===d.split(":")[0]||t.sslForHttp?"https://":"http://",a("mark",["onload",i()]);var e=c.createElement("script");e.src=p.proto+t.agent,c.body.appendChild(e)}}function r(){"complete"===c.readyState&&o()}function o(){a("mark",["domContent",i()])}function i(){return(new Date).getTime()}var a=t("handle"),s=window,c=s.document,f="addEventListener",u="attachEvent",d=(""+location).split("?")[0],p=e.exports={offset:i(),origin:d,features:{}};c[f]?(c[f]("DOMContentLoaded",o,!1),s[f]("load",n,!1)):(c[u]("onreadystatechange",r),s[u]("onload",n)),a("mark",["firstbyte",i()])},{handle:"D5DuLP"}],20:[function(t,e){function n(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(0>o?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=n},{}],21:[function(t,e){function n(t){return!(t&&"function"==typeof t&&t.apply&&!t[i])}var r=t("ee"),o=t(1),i="nr@wrapper",a=Object.prototype.hasOwnProperty;e.exports=function(t){function e(t,e,r,a){function nrWrapper(){var n,i,s,f;try{i=this,n=o(arguments),s=r&&r(n,i)||{}}catch(d){u([d,"",[n,i,a],s])}c(e+"start",[n,i,a],s);try{return f=t.apply(i,n)}catch(p){throw c(e+"err",[n,i,p],s),p}finally{c(e+"end",[n,i,f],s)}}return n(t)?t:(e||(e=""),nrWrapper[i]=!0,f(t,nrWrapper),nrWrapper)}function s(t,r,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<r.length;c++)s=r[c],a=t[s],n(a)||(t[s]=e(a,f?s+o:o,i,s,t))}function c(e,n,r){try{t.emit(e,n,r)}catch(o){u([o,e,n,r])}}function f(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){u([r])}for(var o in t)a.call(t,o)&&(e[o]=t[o]);return e}function u(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=r),e.inPlace=s,e.flag=i,e}},{1:20,ee:"QJf3ax"}]},{},["G9z0Bl",3,10,4]);</script>
        <script>
            
            Date.now = Date.now || function () { return +new Date; };

            if (typeof console === "undefined" || typeof console.log === "undefined") {
                console = {};
                console.log = function () { };
            }

            window.start = Date.now();

            window.diagnostic = function(request_name) {
                var stop = Date.now() - window.start;
                console.log(request_name + ": " + stop);
                if (typeof NREUM === "undefined") {
                    console.log("NREUM not defined");
                } else {
                    if (typeof NREUM.inlineHit === "undefined") {
                        console.log("NREUM.inlineHit not defined");
                    } else {
                        console.log("logging...");
                        NREUM.inlineHit(request_name, 0, 0, 0, 0, stop);
                    }
                }
            };

        </script>

        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />

        
    <title>Frontiers | Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods | Food Microbiology</title>

        
        <link rel="shortcut icon" href="/Images/Frontiers/Common/Icon/favicon.ico" type="image/x-icon" />

        
    <meta property="og:type" content="article" />
    <meta name="Keywords" content="Saccharomyces cerevisiae, Wine, biogeography, molecular methods, terroir" />
    <meta property="og:site_name" name="site_name" content="Frontiers" />
    <meta property="og:title" name="Title" content="Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods" />
    <meta property="og:description" name="Description" content="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the divers..." />
    <meta property="og:url" name="url" content="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/full" />
            <meta name="citation_volume" content="4" />
        <meta name="citation_journal_title" content="Food Microbiology" />
        <meta name="citation_publisher" content="Frontiers" />
        <meta name="citation_journal_abbrev" content="Front. Microbiol." />
        <meta name="citation_doi" content="10.3389/fmicb.2013.00166" />
        <meta name="citation_pages" content="166" />
        <meta name="citation_language" content="English" />
        <meta name="citation_title" content="Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods" />
        <meta name="citation_keywords" content="Saccharomyces cerevisiae; Wine; biogeography; molecular methods; terroir" />
        <meta name="citation_author" content="Tofalo, Rosanna" />
        <meta name="citation_author_institution" content="University of Teramo, Faculty of Bioscienze e Tecnologie Agro alimentari ed Ambientali, Mosciano Sant&#39;Angelo (TE), Italy" />
        <meta name="citation_author_email" content="rtofalo@unite.it" />
        <meta name="citation_author" content="Perpetuini, Giorgia" />
        <meta name="citation_author_institution" content="University of Teramo, Faculty of Bioscienze e Tecnologie Agro alimentari ed Ambientali, Mosciano Sant&#39;Angelo (TE), Italy" />
        <meta name="citation_author_email" content="gperpetuini@unite.it" />
        <meta name="citation_author" content="Schirone, Maria" />
        <meta name="citation_author_institution" content="University of Teramo, Faculty of Bioscienze e Tecnologie Agro alimentari ed Ambientali, Mosciano Sant&#39;Angelo (TE), Italy" />
        <meta name="citation_author_email" content="mschirone@unite.it" />
        <meta name="citation_author" content="Fasoli, Giuseppe" />
        <meta name="citation_author_institution" content="University of Teramo, Faculty of Bioscienze e Tecnologie Agro alimentari ed Ambientali, Mosciano Sant&#39;Angelo (TE), Italy" />
        <meta name="citation_author_email" content="gfasoli@unite.it" />
        <meta name="citation_author" content="Aguzzi, Irene" />
        <meta name="citation_author_institution" content="University of Teramo, Faculty of Bioscienze e Tecnologie Agro alimentari ed Ambientali, Mosciano Sant&#39;Angelo (TE), Italy" />
        <meta name="citation_author_email" content="iaguzzi@unite.it" />
        <meta name="citation_author" content="Corsetti, Aldo" />
        <meta name="citation_author_institution" content="University of Teramo, Faculty of Bioscienze e Tecnologie Agro alimentari ed Ambientali, Mosciano Sant&#39;Angelo (TE), Italy" />
        <meta name="citation_author_email" content="acorsetti@unite.it" />
        <meta name="citation_author" content="Suzzi, Giovanna" />
        <meta name="citation_author_institution" content="University of Teramo, Faculty of Bioscienze e Tecnologie Agro alimentari ed Ambientali, Mosciano Sant&#39;Angelo (TE), Italy" />
        <meta name="citation_author_email" content="gsuzzi@unite.it" />
        <meta name="citation_abstract" content="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the diversity of microorganisms because there would be no way to visualise patterns in variation. Saccharomyces cerevisiae, “the wine yeast”, is the most important species involved in alcoholic fermentation, and in vineyard ecosystems, it follows the principle of “everything is everywhere”. Agricultural practices such as farming (organic versus conventional) and floor management systems have selected different populations within this species that are phylogenetically distinct. In fact, recent ecological and geographic studies highlighted that unique strains are associated with particular grape varieties in specific geographical locations. These studies also highlighted that significant diversity and regional character, or ‘terroir’, have been introduced into the winemaking process via this association. This diversity of wild strains preserves typicity, the high quality and the unique flavour of wines. Recently, different molecular methods were developed to study population dynamics of S. cerevisiae strains in both vineyards and wineries. In this review, we will provide an update on the current molecular methods used to reveal the geographical distribution of S. cerevisiae wine yeast." />
        <meta name="description" content="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the diversity of microorganisms because there would be no way to visualise patterns in variation. Saccharomyces cerevisiae, “the wine yeast”, is the most important species involved in alcoholic fermentation, and in vineyard ecosystems, it follows the principle of “everything is everywhere”. Agricultural practices such as farming (organic versus conventional) and floor management systems have selected different populations within this species that are phylogenetically distinct. In fact, recent ecological and geographic studies highlighted that unique strains are associated with particular grape varieties in specific geographical locations. These studies also highlighted that significant diversity and regional character, or ‘terroir’, have been introduced into the winemaking process via this association. This diversity of wild strains preserves typicity, the high quality and the unique flavour of wines. Recently, different molecular methods were developed to study population dynamics of S. cerevisiae strains in both vineyards and wineries. In this review, we will provide an update on the current molecular methods used to reveal the geographical distribution of S. cerevisiae wine yeast." />
        <meta name="WT.cg_n" content="Frontiers" />
        <meta name="WT.cg_s" content="Science Journals;Health Journals" />
        <meta name="WT.z_cg_cat" content="Food Science;Microbiology;Nutrition" />
        <meta name="WT.z_cg_topic" content="Food Microbiology" />
        <meta name="citation_online_date" content="2013/06/03" />
        <meta name="citation_date" content="2013" />
        <meta name="citation_publication_date" content="2013/06/24" />
        <meta name="citation_fulltext_html_url" content="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/full" />
        <meta name="citation_abstract_html_url" content="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/abstract" />
        <meta name="citation_pdf_url" content="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/pdf" />
        <meta name="fulltext_pdf" content="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/pdf" />
        <meta name="fulltext_html" content="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/full" />
        <meta name="citation_xml_url" content="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/xml" />

        
    <script type="text/javascript">

        var CurrentIBarMenu = 'bysubjects';
        var CurrentPageCode = 'ARTICLE_PAGE_NEW';

        var FRConfiguration = (function() {
            return {
                Environment: 'Live',
                SANVirtualPath: 'http://www.frontiersin.org/files/',
                SharepointWebsiteUrl: 'http://www.frontiersin.org',
                CommunityWebsiteUrl: 'http://community.frontiersin.org',
                FrontiersJournalUIUrl: 'http://journal.frontiersin.org',
                FrontiersJournalAPIUrl: 'https://api-journal.frontiersin.org',
                FrontiersReviewUIUrl: 'http://review.frontiersin.org',
                FrontiersReviewAPIUrl: 'http://api-review.frontiersin.org/v1',
                FrontiersNetworkingAPIUrl: 'http://api-network.frontiersin.org'
            };
        })();

        var FRLanguage = (function() {
            var languageSet = {"ART_FRONTIERS":"Frontiers ","Article_AcceptedDate":"Accepted: ","Article_AnalyticsToolTip":"The total view count is updated once a day, so not to worry if you don\u0027t see immediate results.","Article_AnalyticsTotalViews":"total views","Article_AnalyticsViewImpact":"View Article Impact","Article_ArchiveLinkText":"Archive","Article_BibTex":"BibTex","Article_Citation":"Citation: ","Article_Commentary":"COMMENTARY","Article_Copyright":"Copyright: ","Article_CopyrightText":"This is an open-access article distributed under the terms of the \u003ca href=\"http://creativecommons.org/licenses/by/4.0/\"\u003eCreative Commons Attribution License (CC BY)\u003c/a\u003e. The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.","Article_Correspondence":"* Correspondence: ","Article_DownloadPDF":"Download PDF","Article_EditedBy":"Edited by: ","Article_EndNote":"EndNote","Article_ExportCitation":"Export Citation","Article_JATS":"JATS","Article_Keywords":"Keywords: ","Article_NLM":"NLM","Article_OriginalArticle":"ORIGINAL ARTICLE","Article_PaperPendingPublishedDate":"Paper pending published: ","Article_PDF":"PDF","Article_ProvisionalPDF":"Provisional PDF","Article_PublishedDate":"Published online: ","Article_ReadFullText":"Read Full Text","Article_ReceivedDate":"Received: ","Article_ReferenceManager":"Reference Manager","Article_ReviewedBy":"Reviewed by: ","Article_RTInfoText":"This article is part of the Research Topic","Article_ShareOn":"SHARE ON","Article_SimpleTEXTfile":"Simple TEXT file","Article_SupplementalData":"SUPPLEMENTAL DATA","Article_TableOfContent":"TABLE OF CONTENTS","Article_ViewEnhancedPDF":"View Enhanced PDF","Article_XML":"XML","BrowserWarningText":"\u003ch2\u003eWarning!\u003c/h2\u003e\u003cp\u003eYou are using an \u003cstrong\u003eoutdated\u003c/strong\u003e browser. This page doesn\u0027t support Internet Explorer 6, 7 and 8.\u003cbr /\u003ePlease \u003ca class=\"blue\" href=\"http://browsehappy.com/\"\u003eupgrade your browser\u003c/a\u003e or \u003ca class=\"blue\" href=\"http://www.google.com/chromeframe/?redirect=true\"\u003eactivate Google Chrome Frame\u003c/a\u003e to improve your experience.\u003c/p\u003e","COMMENT_HEADERTEXT":"Comment text too long","COMMENT_WARNINGTEXT":"Comments must be less than 4,000 characters. You have entered ","Impact_BackToArticle":"Back to article","People_Also_LookedAt":"People also looked at"};

            return {
                value: function(key) {
                    if (languageSet[key]) {
                        return languageSet[key];
                    } else {
                        throw new Error('Unable to get the value status from the language set'); // Use Error, not FRError
                    }
                }
            };

        })();

    </script>

    <script src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-535f7e222fc3ca72"></script>

    <script language="javascript">
        var addthis_config = addthis_config || {};
        addthis_config.data_track_addressbar = false;
        addthis_config.data_track_clickback = false;
    </script>
    
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
    <script src="https://maps.googleapis.com/maps/api/js?sensor=false"></script>

    <script src="http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/src/data.json"></script>

    <script src="http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/src/markerclusterer.js"></script>






        <link href="/bundles/frontiers-basic-css?v=mtPmZlF0BWtgds-7X3Jz3E8Orup7OjMPKxP8YyUxKaw1" rel="stylesheet"/>

        <link href="/bundles/frontiers-common-css?v=6kyYHTZLW68OMECRwzyMCPY9wZRIU8MV9rSsHinErP41" rel="stylesheet"/>


        
    <link href="/bundles/frontiers-journal-css?v=mGFzI9kkja-ctZVG3E4d8moVt6porocLyjDlCqMGATo1" rel="stylesheet"/>

    

    </head>
    <body>
        <div class="ibar"></div>

<div class="container alert-box">
    <div class="alert-content">
        <ul id="alerts"></ul>
    </div>
</div>


        





<!--[if lte IE 8]>
    <div class="row">
        <div class="span16">
                    <div style="margin-bottom: 15px;"><h2>Warning!</h2><p>You are using an <strong>outdated</strong> browser. This page doesn't support Internet Explorer 6, 7 and 8.<br />Please <a class="blue" href="http://browsehappy.com/">upgrade your browser</a> or <a class="blue" href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p></div>
        </div>
    </div>
<![endif]-->

<div class="page-container">
    <div class="container">
        


<div class="article-section">
    
    <div class="row">
    <div class="span16">
        <div class="element banner-wrapper"></div>
    </div>
</div>
<div class="row">
    <div class="span16 article-main-bar">
        <div class="row">
            <div class="span7">
                <div class="back-page">
                    <a href="http://www.frontiersin.org/Journal/Archive.aspx?s=441&amp;name=Food Microbiology"><span class="fr-icon-v2-left-open"></span>Archive</a>
                </div>
            </div>
        
                <div class="span9">
                    <div class="element research-topic-block">
                        <div class="thumbnail-icon">
                            <i class="fr-icon-medium research-topic"></i>
                        </div>
                        <div class="research-topic-data">
                            <span class="topic-title">This article is part of the Research Topic</span>
                            <a href="http://www.frontiersin.org/Food_Microbiology/researchtopics/Molecular_methods_in_food_qual/570" data-topic="Molecular methods in food quality and safety" class="topic-link-trim"   data-webtrendtracking='{ "WT_action": "research_topics_click", "WT_destination": "research_topics_page", "WT_source": "article", "WT_dl": "1", "WT_ndl": "1"  }'>Molecular methods in food quality and safety</a>
                        </div>
                    </div>
                </div>
        </div>
    </div>
</div>

    <div class="row">
        <div class="span16">
            <div class="row article-container">
                <div class="span12 abstract-container">
                    <div class="article-header-container">
                        <div class="header-bar-one">
                            <h2>Review ARTICLE</h2>
                        </div>
                        <div class="header-bar-three">
                            Front. Microbiol., 24 June 2013 | doi: 10.3389/fmicb.2013.00166
                        </div>
                    </div>
<div class="JournalAbstract">
<a id="h1" name="h1"></a><h1>Biogeographical characterization of <i>Saccharomyces cerevisiae </i>wine yeast by molecular methods</h1>
<div class="authors">
<a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=RosannaTofalo&#x00026;UID=45789">Rosanna Tofalo</a>*, <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=GiorgiaPerpetuini&#x00026;UID=58814">Giorgia Perpetuini</a>, <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=MariaSchirone&#x00026;UID=36163">Maria Schirone</a>, <a href="http://community.frontiersin.org/people/GIUSEPPEFASOLI/98453">Giuseppe Fasoli</a>, <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=IreneAguzzi&#x00026;UID=51168">Irene Aguzzi</a>, <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=AldoCorsetti&#x00026;UID=33275">Aldo Corsetti</a> and <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=GiovannaSuzzi&#x00026;UID=18039">Giovanna Suzzi</a>*</div>
<ul class="notes">
<li class="pl0">Faculty of BioScience and Technology for Food, Agriculture and Environment, University of Teramo, Mosciano Sant&#x02019;Angelo, TE, Italy</li>
</ul>
<p>Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the diversity of microorganisms because there would be no way to visualize patterns in variation. <i>Saccharomyces cerevisiae</i>, &#x0201C;the wine yeast,&#x0201D; is the most important species involved in alcoholic fermentation, and in vineyard ecosystems, it follows the principle of &#x0201C;everything is everywhere.&#x0201D; Agricultural practices such as farming (organic <i>versus</i> conventional) and floor management systems have selected different populations within this species that are phylogenetically distinct. In fact, recent ecological and geographic studies highlighted that unique strains are associated with particular grape varieties in specific geographical locations. These studies also highlighted that significant diversity and regional character, or &#x02018;<i>terroir</i>,&#x02019; have been introduced into the winemaking process via this association. This diversity of wild strains preserves typicity, the high quality, and the unique flavor of wines. Recently, different molecular methods were developed to study population dynamics of <i>S. cerevisiae</i> strains in both vineyards and wineries. In this review, we will provide an update on the current molecular methods used to reveal the geographical distribution of<i> S. cerevisiae </i>wine yeast.</p>
<div class="clear"></div>
</div>
<div class="JournalFullText">
<a id="h2" name="h2"></a><h2>Introduction</h2>
<p class="mb15">One of the most important issues in ecological studies is the determination of microbial biodiversity distribution and thus the understanding of whether microorganisms are cosmopolitan or endemic to a specific area or host (<a href="#B101">Ramette and Tiedje, 2007</a>). Biogeography is the discipline that studies the distribution of biodiversity over space and time (<a href="#B78">Martiny et al., 2006</a>). During the 18<sup>th</sup> century, biologists applied this approach to study the geographic distribution of plant and animal diversity, and only more recently, interest in the geographic distribution of microorganisms has increased. The aim of microbiogeography is to reveal where microorganisms live, their abundance and distribution, and their diversity over different taxonomic and spatial scales. In fact, genetic distance may be correlated with geographic distance and/or environmental characteristics (e.g., salinity, depth, latitude; <a href="#B112">Schuller et al., 2012</a>). The scope of microbiogeography also encompasses the understanding of the processes generating and maintaining the distribution of microorganisms (<a href="#B101">Ramette and Tiedje, 2007</a>). Other goals of this field are to propose and evaluate theories regarding the creation and evolution of such diversity patterns in the environment (<a href="#B101">Ramette and Tiedje, 2007</a>). The first paradigm in microbial biogeography, &#x0201C;<i>Alles is overal, maar het milieu selecteert</i>&#x0201D; (&#x0201C;everything is everywhere, but the environment selects&#x0201D;) was offered by Baas Becking more than 70 years ago (<a href="#B94">O&#x02019;Malley, 2008</a>). This appealing idea was based on the small size and high dispersal potential of microorganisms and their large populations and low presumed extinction rates (<a href="#B101">Ramette and Tiedje, 2007</a>). However, even if the field of microbial biogeography is not new, the determinism of microbial diversification and distribution has been poorly documented and is not well understood. This may partly be due to the natural properties of microorganisms (e.g., their small size, which makes access within different environmental matrices difficult, their huge diversity, and the complexity of precisely defining their species) and the lack of an adequate sampling strategy. Recently, the development of new molecular tools has partially resolved these limitations; in fact, recent developments have allowed the survey of uncultivated microorganisms in the environment and the characterization of microbial community structure (<a href="#B24">Christen, 2008</a>). Furthermore, these tools are now generally automated and allow the moderate throughput essential to studies involving the characterization of numerous samples of different origins. In particular, the use of DNA, RNA and protein sequences for the construction of evolutionary trees has allowed a better understanding of the way in which biodiversity was generated. Hence, the application of molecular phylogenetic methods to study natural microbial ecosystems has resulted in the unexpected discovery of many evolutionary lineages (<a href="#B123">Suzzi, 2011</a>). Moreover, metagenomic and metatranscriptomic approaches will allow not only the dissolution of the species concept issue but will also separate the relationship between the notion of species and their spatial distribution. <a href="#B137">Weiher and Keddy (1995)</a> proposed that a trait-based approach should be the basis of a conceptual model for trait-based community assembly. In particular, traits, not taxon names, are the fundamental units of biodiversity and biogeography. Microorganisms that show similar traits share the same ecological niche. Therefore, the principal challenge of microbiology is to identify the main genetic variants inducing phenotypic variation and niche adaptation (<a href="#B30">Cubillos et al., 2011</a>).</p>
<p class="mb0">Yeasts of the <i>Saccharomyces sensu stricto</i> species complex (Figure <a href="#F1">1</a>) are able to convert sugar into ethanol and CO<sub>2</sub> via fermentation. They have been used for thousands of years by mankind for the production of fermented beverages and foods. These yeasts show interesting features that are specific and not found in other genera; for example, they are able to survive in the absence of oxygen by using the fermentation process (<a href="#B117">Sicard and Legras, 2011</a>). The <i>Saccharomyces sensu stricto</i> genus is composed of species showing a level of nucleotide divergence similar to that found between birds and humans (<a href="#B35">Dujon, 2006</a>). The <i>sensu stricto</i> complex is thought to be young; in fact, some studies have suggested that <i>Saccharomyces</i> <i>cerevisiae</i> diverged from the common ancestor of <i>Saccharomyces</i> <i>paradoxus</i> and <i>Saccharomyces</i> <i>cariocanus </i>approximately 5&#x02013;10 million years ago (Mya), whereas <i>Saccharomyces</i> <i>kudriavzevii. Saccharomyces</i> <i>bayanus</i>, and <i>Saccharomyces</i> <i>mikatae</i> diverged 10&#x02013;15, 15&#x02013;20, and 20 Mya, respectively (reviewed in <a href="#B103">Replansky et al., 2008</a>).</p>
<div class="DottedLine"></div>
<div class="Imageheaders">FIGURE 1</div>
<div class="FigureDesc">
<a href="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_m/fmicb-04-00166-g001.jpg" name="figure1" target="_blank">
<img src="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_t/fmicb-04-00166-g001.gif" id="F1" alt="www.frontiersin.org" /></a>
<p><strong>FIGURE 1. Geographical characteristics and phylogenetic relationship among <i>Saccharomyces</i> species based on the combined sequence analysis of the D1/D2 LSU rRNA gene and ITS (modified from <a href="#B103">Replansky et al., 2008</a>; <a href="#B60">Kurtzman et al., 2011</a>).</strong></p>
</div>
<div class="clear"></div>
<div class="DottedLine"></div>
<p class="mb15 w100pc float_left mt15">Recently, <a href="#B69">Libkind et al. (2011)</a> identified a new species very similar to <i>Saccharomyces</i> <i>bayanus </i>and called it <i>Saccharomyces eubayanus</i> sp. nov., which exists in apparent sympatry in <i>Nothofagus</i> (Southern beech) forests in Patagonia. This species is 99.5% identical to the non-<i>Saccharomyces cerevisiae</i> portion of the <i>Saccharomyces</i> <i>pastorianus </i>genome sequence. Because <i>Saccharomyces pastorianus</i> and <i>Saccharomyces bayanus </i>(a complex hybrid of <i>Saccharomyces eubayanus</i>, <i>Saccharomyces uvarum</i>, and <i>Saccharomyces cerevisiae</i>) are considered to be &#x0201C;a product of the artificial brewing environment with no occurrence in nature,&#x0201D; they may be associated with domestication events and hybrid lineages, whereas <i>Saccharomyces</i> <i>uvarum</i> and <i>Saccharomyces</i> <i>eubayanus</i> may be conserved as descriptors of the species.</p>
<p class="mb15">Recently, the budding yeast <i>S</i>. <i>cerevisiae </i>has been considered to be an important model for ecological and evolutionary genetics. The ancestor of the <i>sensu stricto</i> complex underwent whole-genome duplication. This event was followed by the loss of approximately 90% of the duplicated genes. In fact, comparison of the <i>S</i>. <i>cerevisiae</i> genome with that of the pre-duplication species <i>Kluyveromyces waltii</i> reveals the presence of approximately 500 paralogs among the 5500 genes (<a href="#B103">Replansky et al., 2008</a>). These duplications may then be subjected to mutations, which may be related to the evolution of new functions or sequence divergence, and inactivation due to the accumulation of non-sense mutations, which leads to relics (<a href="#B72">Liti and Louis, 2005</a>). The duplicated genes may evolve at different rates, providing new functions. For example, the abilities to grow anaerobically and to produce ethanol and the low- and high-affinity glucose systems may be a consequence of genome duplication and may have offered a competitive advantage against bacteria and other microorganisms.</p>
<p class="mb15"><a href="#B109">Scannell et al. (2011)</a> resequenced and reassembled the genomes of <i>S</i>. <i>mikatae</i>, <i>S</i>. <i>kudriavzevii</i>, and <i>S</i>. <i>bayanus</i> and compared them with the <i>S</i>. <i>paradoxus</i> genome (<a href="#B71">Liti et al., 2009</a>) and the reference genome of <i>S</i>. <i>cerevisiae</i> (<a href="#B50">Goffeau et al., 1996</a>). The authors annotated 5261 sets of genes that are orthologous among all five species and identified 123 genes that could be used as targets of positive selection and may play important roles in ecological specialization. Moreover, these authors underlined that whole-genome duplication still influences yeast evolution and contributes to the genomic and phenotypic differences characterizing <i>S</i>. <i>cerevisiae </i>and its related species. In addition, the presence of two possible horizontal gene transfers from bacteria was described (<a href="#B109">Scannell et al., 2011</a>). The possibility of a horizontal genetic exchange from bacteria was also suggested by <a href="#B136">Wei et al. (2007)</a>. The YJM789 genome (a yeast isolated from the lung of an AIDS patient with pneumonia) highlighted a putative horizontal transfer of YJM-GNAT (an unknown gene belonging to the GNAT superfamily related to antibiotic resistance) from bacteria and a potential introgression of a 12-kb sequence of chromosome I from a closely related yeast (<a href="#B136">Wei et al., 2007</a>).</p>
<p class="mb15">Recent analyses have shown that yeast hybrids may be more abundant in both natural and industrial environments than previously thought. Indeed, almost 10% of <i>Saccharomyces</i> strains previously classified as <i>sensu stricto </i>appear to be hybrids of different species (<a href="#B72">Liti and Louis, 2005</a>). In fact, interspecific hybrid strains, which contain genetic contributions from both <i>S. cerevisiae </i>and other <i>Saccharomyces</i> spp., may have selective advantages deriving from the combination of desirable traits from both parental species. Recently, several strains involved in winemaking were found to be hybrids between <i>S</i>. <i>cerevisiae</i> and <i>S. kudriavzevii</i> (<a href="#B51">Gonzales et al., 2006</a>; <a href="#B41">Erny et al., 2012</a>). Initially, this last species was isolated in Japan, and although <a href="#B108">Sampaio and Goncalves (2008)</a> also found it in Portugal, it has never been isolated from wine fermentation. However, the Portuguese <i>S. kudriavzevii </i>population showed genetic differences compared with the type strain of the species that represents the Japanese population. In wine fermentation, the hybrids exhibit the best properties of both parental species, such as the low-temperature fermentation ability of <i>S. kudriavzevii </i>and the high ethanol resistance of <i>S</i>. <i>cerevisiae</i>. <a href="#B37">Dunn et al. (2012)</a> analyzed 69 commercial wine yeasts and compared them with other industrial yeasts, wine yeasts, beer yeasts, bread yeasts, and fuel ethanol yeasts. An interspecific hybridization between <i>S. cerevisiae </i>and <i>S. kudriavzevii </i>in four of the 69 commercial wine strains was observed, and <i>S. paradoxus </i>and <i>S. mikatae </i>introgression events were detected.</p>
<p class="mb15">It is unknown when humans began to add selected yeast to make fermented beverages and foods. Such human activities caused hybridization between species and variation of ploidy, which contributed to the evolution of domesticated yeasts.</p>
<p class="mb15"><i>Saccharomyces</i> <i>cerevisiae</i> strains are adapted to different niches, so they represent a rich resource for revealing the evolutionary trajectories of a trait because particular molecular profiles may have been selected in specific environments. Moreover, several studies found evidence for a role of geographical isolation in the differentiation of the <i>S</i>. <i>cerevisiae</i> population in nature, indicating that <i>S</i>. <i>cerevisiae</i> can be used as model for evolutionary biology and biogeography (<a href="#B18">Carreto et al., 2008</a>; <a href="#B103">Replansky et al., 2008</a>; <a href="#B70">Lidzbarsky et al., 2009</a>; <a href="#B71">Liti et al., 2009</a>).</p>
<p class="mb15">Recent resequencing and phylogenetic characterization of multiple <i>S. cerevisiae</i> isolates provided evidence of substantial genetic and phenotypic diversity (<a href="#B71">Liti et al., 2009</a>).</p>
<p class="mb15"><a href="#B134">Wang et al. (2012)</a> performed a population genetics analysis of wild Chinese isolates with different ecological and geographical origins. They identified eight new, distinct wild lineages (coded as CHNI-VIII) from a set of 99 Chinese isolates. These lineages were characteristic of specific geographical areas and ecological niches. In particular, these results indicate that a geographically isolated source is important for <i>S</i>. <i>cerevisiae</i> population differentiation in nature. In fact, this study showed that oak isolates from different regions in northern China clustered into different lineages, and the Chinese oak isolates were clearly separated from those from North America.</p>
<p class="mb15">Strains of <i>S. cerevisiae</i> associated with vineyards and wine production, hereafter referred to as wine strains, often form a genetically differentiated group that is separate from wild strains isolated from soil and oak tree habitats and strains from other fermentation types, such as palm wine, and sake (<a href="#B42">Fay and Benavides, 2005</a>; <a href="#B64">Legras et al., 2007</a>; <a href="#B71">Liti et al., 2009</a>; <a href="#B110">Schacherer et al., 2009</a>). Several authors have explained these differences as a consequence of domestication. These domestication events were followed by human-associated dissemination of these yeasts throughout the world.</p>
<p class="mb15">In a recent study, <a href="#B64">Legras et al. (2007)</a> investigated the possible effects of human history on spreading and selecting this yeast. In particular, they analyzed 651 yeast strains with 56 different origins (beer, bread, palm wine, wine, and rice wine) from five continents. All wine yeasts grouped together and were well separated from the yeast strains of other technological origins. For &#x0201C;non-wine strains,&#x0201D; a relationship between genotype and the isolation source was found. In particular, three Asian groups of strains were identified: the first included the sake yeast group, and the other two contained rice wine and Chinese distillery strains. Regarding the African yeast populations, a Nigerian palm wine group was identified, which also included an Ivory Coast strain. Ghana sorghum beer strains were distinct from palm wine, Burundi cassava and banana strains, suggesting genetic differentiation among African yeast populations (<a href="#B64">Legras et al., 2007</a>). The wine yeast group contains strains from ancient vine areas (Lebanon, Europe) as well as &#x02018;new world&#x02019; recent vineyards, which suggests a migration of wine yeast all over the world. In addition to the historical human transport across the Mediterranean Sea, the phylogenetic analysis obtained clearly supports the hypothesis of a migration pathway along the Danube valley. The way in which wine strains are naturally propagated is still poorly understood: flor yeasts, which grow almost continuously on the surface of wine during the sherry wine process, are likely an example of domestication; in fact, they may present specific features and mutations (<a href="#B43">Fidalgo et al., 2006</a>). However, for other types of wine strains, we cannot infer such a continuous human control of their culture.</p>
<p class="mb15"><a href="#B42">Fay and Benavides (2005)</a> investigated the genetic differences among strains that were of wine origin and those that were not. The population of <i>S</i>. <i>cerevisiae</i> associated with the wine ecosystem were genetically homogenous. The reduced levels of variation present in winemaking strains may have been the result of a genetic bottleneck, selection for specific traits, or a combination of the two (<a href="#B56">Hyma et al., 2011</a>).</p>
<p class="mb0">The population structure of <i>S</i>. <i>cerevisiae</i> in nature remains obscure. The different <i>S</i>. <i>cerevisiae</i> isolates are characterized by large genetic and phenotypic variations, providing a powerful tool for quantitative genetic studies (<a href="#B71">Liti et al., 2009</a>). This apparent variation is likely because some studies were performed on laboratory strains, which are highly adapted to artificial conditions and do not represent the true ecological diversity of the species (<a href="#B122">Steinmetz et al., 2002</a>; <a href="#B40">Ehrenreich et al., 2010</a>). Recently, it was demonstrated that environmental factors and the interactions between each organism and its environment influence genomic rearrangements and the evolution of phenotypes (<a href="#B17">Camarasa et al., 2011</a>; <a href="#B135">Warringer et al., 2011</a>). In particular, <a href="#B17">Camarasa et al. (2011)</a> related the metabolic traits of <i>S. cerevisiae </i>strains with their origins. These strains were isolated from seven different niches (baker, clinical, fermentation processes, laboratory, vineyard, natural, and commercial wine yeasts). The relationships were established using a statistical approach that allowed the identification of specific features common to all strains belonging to the same niche. Some metabolic differences of strains with different origins are shown in Table <a href="#T1">1</a>. Phenotypic variation in <i>Saccharomyces</i> strains collected from diverse natural habitats, used in industrial processes, and associated with human illness was observed (<a href="#B61">Kvitek et al., 2008</a>). Phenotypic variation in stress sensitivity and gene expression was also observed. Vineyard isolates survived better in the presence of different stress conditions due to their ability to thrive in more variable natural environments, which facilitated their dispersal into new environments in a manner associated with human interactions (<a href="#B61">Kvitek et al., 2008</a>). The main approach used to establish a relationships between genotypic variation and phenotypes is mapping of quantitative trait loci (QTL). This technique allowed to map the loci responsible for brewing characteristics in a sake strain, ethanol resistance, xylose utilization for application in the bioethanol industry, acetic acid production, and fermentation performance in wine strains (<a href="#B13">Borneman et al., 2013</a>).</p>
<div class="DottedLine"></div>
<div class="Imageheaders">TABLE 1</div>
<div class="FigureDesc">
<a href="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_m/fmicb-04-00166-t001.jpg" name="table1" target="_blank">
<img src="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_t/fmicb-04-00166-t001.gif" id="T1" alt="www.frontiersin.org" /></a>
<p><strong>TABLE 1. Sequenced genomes of wine <i>Saccharomyces cerevisiae</i> strains (modified from <a href="#B13">Borneman et al., 2013</a>).</strong></p>
</div>
<div class="clear"></div>
<div class="DottedLine"></div>
<p class="mb15 w100pc float_left mt15">It is well known that the wine yeast, <i>S. cerevisiae, </i>plays a major role in the fermentation of grape musts; in fact, it is well adapted to this process (<a href="#B77">Martini and Vaughan-Martini, 1990</a>; <a href="#B9">Blondin et al., 2011</a>). In particular, this yeast is adapted to the harsh conditions in grape musts and grapes (high sugar concentration, increasing alcohol concentration, acidity, presence of sulfites, anaerobiosis, and progressive depletion of essential nutrients, such as nitrogen, vitamins, and lipids), and its genome has been modeled, so the understanding of the adaptation phenomenon to the wine environment is a key element in wine yeast genome research (<a href="#B9">Blondin et al., 2011</a>).</p>
<a id="h3" name="h3"></a><h2><i>Saccharomyces cerevisiae</i> Wine Yeast</h2>
<p class="mb0"><i>Saccharomyces cerevisiae</i> is one of the best model systems used for understanding microbial ecology and evolutionary genetics. Many functional analysis projects have been dedicated to the investigation of its molecular biology since its genome was first sequenced more than 10 years ago. In fact, a large amount of genomic data for <i>S</i>. <i>cerevisiae</i> strains is available (<a href="#B136">Wei et al., 2007</a>; <a href="#B93">Novo et al., 2009</a>; <a href="#B11">Borneman et al., 2011</a>, <a href="#B13">2013</a>): there are 28 assembled genome sequences (mainly in draft format), and 19 are available as unassembled sequencing reads. Moreover, 35 sequences are available through project-specific websites (<a href="#B13">Borneman et al., 2013</a>). In Table <a href="#T2">2</a>, wine <i>S</i>. <i>cerevisiae </i>sequenced strains are reported.</p>
<div class="DottedLine"></div>
<div class="Imageheaders">TABLE 2</div>
<div class="FigureDesc">
<a href="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_m/fmicb-04-00166-t002.jpg" name="table2" target="_blank">
<img src="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_t/fmicb-04-00166-t002.gif" id="T2" alt="www.frontiersin.org" /></a>
<p><strong>TABLE 2. Some traits of <i>Saccharomyces cerevisiae</i> strains from different origins.</strong></p>
</div>
<div class="clear"></div>
<div class="DottedLine"></div>
<p class="mb15 w100pc float_left mt15">Recently, <a href="#B10">Borneman et al. (2012)</a> described the genome sequence of the thiol-releasing commercial wine yeast hybrid VIN7. They showed that VIN7 is an almost complete allotriploid interspecific hybrid of <i>S. cerevisiae</i> and <i>S. kudriavzevii</i> that contains a heterozygous diploid <i>S. cerevisiae</i> genome and a haploid <i>S. kudriavzevii</i> genome. Both parental strains showed a European origin; in particular, the <i>S. cerevisiae</i> portion of the VIN7 genome was closely related to wine yeast but distant from the commercial wine yeasts QA23 and EC118 (<a href="#B10">Borneman et al., 2012</a>). The genomes of <i>S</i>. <i>cerevisiae</i>/<i>S</i>. <i>kudriavzevii</i> hybrid strains display a mosaic structure that likely resulted from selective pressures experienced over time (<a href="#B98">Querol and Bond, 2009</a>).</p>
<p class="mb15">A comparative genome analysis between <i>S. cerevisiae</i> industrial and laboratory strains highlighted how the environment influences genomic structure and helped to identify genomic loci involved in the regulation of industrial phenotypes. In particular, substantial conservation throughout a core set of genes was observed, whereas many other regions displayed nucleotide substitutions likely involved in diversification and specialization events (<a href="#B12">Borneman et al., 2008</a>).</p>
<p class="mb15">Gene transfer is an important aspect of yeast diversification and may play a major role in adaptation to the wine fermentation ecosystem. <a href="#B93">Novo et al. (2009)</a> sequenced the complete genome of the diploid commercial wine yeast EC118. They identified 34 ORFs encoding proteins potentially involved in carbon and nitrogen metabolism, cellular transport, and the stress response that were absent from S288c. BLASTP analysis suggested that these genes specific for EC1118 were acquired from non-<i>S. cerevisiae</i> donors. In fact, the closest relatives to EC1118 were found to be in species belonging to two clades. The first contained the <i>Lachancea</i>, <i>Zygosaccharomyces</i>, <i>Kluyveromyces</i>, <i>Saccharomyces</i>, and <i>Eremothecium</i> genera, and the second species belonged to a large, recently reassessed clade containing <i>Debaryomyces</i>, some <i>Pichia</i>, and a number of medically important <i>Candida</i> species.</p>
<p class="mb15">The yeast genome is quite small at only 12 Mb but is highly packed, with approximately 6000 genes distributed over 16 chromosomes. Additionally, it contains two small, cytoplasmatic genomes: mitochondrial DNA (mtDNA) and killer double-stranded RNA (dsRNA). The biological and genetic characteristics of <i>S</i>. <i>cerevisiae</i> have been recently reviewed by <a href="#B62">Landry et al. (2006)</a>. Briefly, <i>S</i>. <i>cerevisiae</i> is a diploid yeast with highly clonal reproduction. <i>S. cerevisiae</i> is also homothallic, which confers the ability of regenerating a diploid cell from a haploid and could be interpreted as a way of genome renewal. This mechanism may be responsible for the high rate (28%) of homozygote strains found in vineyards (<a href="#B83">Mortimer et al., 1994</a>). Many studies have also described the aneuploidy of wine (<a href="#B4">Bakalinsky and Snow, 1990</a>; <a href="#B53">Guijo et al., 1997</a>; <a href="#B87">Nadal et al., 1999</a>), beer or bread strains (<a href="#B26">Codon et al., 1998</a>).</p>
<p class="mb15">The story of <i>S. cerevisiae</i> populations on earth is lost in the mists of history, and despite over 70 years of research, the biogeography of <i>S</i>. <i>cerevisiae</i> remains elusive; in fact, little is known about its ecology, origin, evolution, and distribution in nature (<a href="#B134">Wang et al., 2012</a>). <a href="#B90">Naumov et al. (2006)</a> hypothesized that the most ancient population of <i>S</i>. <i>cerevisiae </i>originated from Malaysia. Ancient <i>S</i>. <i>cerevisiae</i> DNA was discovered in Chinese pottery jars (7.000&#x02013;5.500 B.C.; <a href="#B80">McGovern et al., 2004</a>; <a href="#B121">Stefanini et al., 2012</a>). Moreover, ribosomal DNA from <i>S</i>. <i>cerevisiae</i> was also found in some wine jars in the King Scorpion tomb in Abydos, Egypt, indicating that this yeast was responsible for wine fermentation by at least 3150 B.C. (<a href="#B22">Cavalieri et al., 2003</a>).</p>
<p class="mb15">During the last 30 years, a large number of observations have demonstrated that the wine strains of <i>S. cerevisiae</i> are highly diverse. Thus, the occurrence of specific natural strains likely depends on numerous factors such as climate conditions, the geographical location of the vineyard, the ripeness of the grapes, the age of the vineyard, the soil type, the grape variety, the application of antifungals, and the technique used to harvest (<a href="#B27">Combina et al., 2005</a>; <a href="#B129">Valero et al., 2005</a>, <a href="#B128">2007</a>; <a href="#B102">Raspor et al., 2006</a>; <a href="#B92">Nisiotou and Nychas, 2007</a>; <a href="#B23">Chavan et al., 2009</a>; <a href="#B68">Li et al., 2010</a>; <a href="#B29">Cordero-Bueso et al., 2011</a>). In fact, another study found a relationship between specific natural strains and a particular <i>terroir </i>(<a href="#B47">Frezier and Dubourdieu, 1992</a>; <a href="#B107">Sabate et al., 1998</a>; <a href="#B74">Lopes et al., 2002</a>; <a href="#B111">Schuller et al., 2005</a>; <a href="#B128">Valero et al., 2007</a>). Thus, the following definition of vitivinicultural &#x0201C;terroir&#x0201D; was provided: &#x0201C;<i>Vitivinicultural &#x0201C;terroir&#x0201D; is a concept which refers to an area in which collective knowledge of the interactions between the identifiable physical and biological environment and applied vitivinicultural practices develops, providing distinctive characteristics for the products originating from this area</i>&#x0201D; (Resolution OIV/Viti 333/2010).</p>
<p class="mb15">However, insufficient quantitative data are available to establish general conclusions on the influence of these factors on the evolution of the fermentative biota of a given viticultural region, and extensive biogeographical surveys over many years are necessary (<a href="#B113">Schuller and Casal, 2007</a>). Discrimination at the strain level thus becomes a strategic activity for the wine industry because it may link territory, environment, and final products for wine valorisation.</p>
<p class="mb15">The species present on intact, undamaged berries have been reported to mainly belong to the group of oxidative basidiomycetous yeasts such as <i>Hanseniaspora uvarum</i>, <i>Cryptococcus</i> spp., <i>Rhodotorula</i> spp., <i>Sporobolomyces</i> spp., and <i>Filobasidium</i> spp. as well as to the dimorphic ascomycetous black yeast <i>Aureobasidium pullulans</i> (<a href="#B97">Prakitchaiwattana et al., 2004</a>; <a href="#B5">Barata et al., 2008</a>, <a href="#B6">Barata et al., 2012</a>). In contrast, the most relevant fermentative wine yeast, <i>S. cerevisiae</i>, only occurs at concentrations less than 10&#x02013;100 cfu/g of berry (<a href="#B44">Fleet, 2003</a>). This yeast is present in nature at very low concentrations. On the surface of undamaged berries, its concentration is lower than 0.1%, although it is easily found on berries damaged by birds or insects (24%), which represent approximately 1 in 1000 grapes. In any case, some authors (<a href="#B82">Mortimer and Polsinelli, 1999</a>) have shown that a population of yeast that is the primary source of natural yeast in wine production exists on grapes. Moreover, data indicate that yeast populations on wine grapes increase from 10<sup>2</sup>&#x02013;10<sup>3</sup> cfu/g on immature berries to 10<sup>3</sup>&#x02013;10<sup>6</sup> cfu/g on mature berries. Insects and birds are important agents for the dispersal of yeasts in different habitats. Regarding the role of insects as a vector for <i>S. cerevisiae</i> cells, <a href="#B82">Mortimer and Polsinelli (1999)</a> demonstrated the presence of a flow of <i>S. cerevisiae</i> cells between the natural environment and cellars; because this yeast is not an airborne, it needs a vector to move. In particular, <a href="#B45">Francesca et al. (2012)</a> also highlighted that migratory birds may act as vector for <i>S. cerevisiae </i>cells, but they are not a &#x0201C;reservoir&#x0201D; because the yeast cells survive in the gut for only 12 h. <a href="#B121">Stefanini et al. (2012)</a> isolated yeast strains from wasps, grapes, and fermentations from the same vineyard over a span of different months and years. The results obtained showed that these strains were more similar to each other than strains derived from other environmental and geographical locations. Wasps therefore may play a role in maintaining ecological diversity.</p>
<p class="mb0">However, whether these strains participate in alcohol fermentation in the cellar is still controversial: some authors (<a href="#B25">Ciani et al., 2004</a>) observed that only cellar strains were responsible for alcohol fermentation in vats, whereas others showed that &#x02018;grapevine strains&#x02019; may be partially responsible for alcohol fermentation (<a href="#B28">Constanti et al., 1997</a>; <a href="#B54">Guti&#x000E9;rrez et al., 1999</a>; <a href="#B66">Le Jeune et al., 2006</a>).</p>
<a id="h4" name="h4"></a><h2>Molecular Methods</h2>
<p class="mb15">By using the techniques developed by molecular geneticists, new phylogenetic relationships were recognized, the number of separate species groups was reduced, and the diversity within the groups was increased (<a href="#B79">McCullough et al., 1998</a>). Moreover, molecular methods revived the study of biogeography and positively impacted the final interpretation of biogeographic patterns (<a href="#B101">Ramette and Tiedje, 2007</a>). In particular, a better knowledge of the microbial ecology of local ecosystems is essential to understand the winemaking process and to generate products with a local character, thereby allowing the development of modern winemaking practices and the diversification of wine products. Grapevine cultivation and wine production spread throughout the Mediterranean Sea toward Greece (5000 B.C.), Italy (900 B.C.), France (600 B.C.), northern Europe (100 AD) and much later, to the Americas (1500 AD). There are approximately 7.5 million hectares of vineyards across the world, mainly concentrated within the earth&#x02019;s temperate zones, and two million are located in Europe (OIV, Statistical Report on World Vitiviniculture, 2012).</p>
<p class="mb15">In particular, many molecular methods allow the identification of <i>S. cerevisiae </i>at the strain level, and they are required not only to investigate the diversity of this species but also to select strains for use as pure cultures, a widespread practice in winemaking industries where strains contribute to a specific characteristic of the final product (<a href="#B32">Dequin, 2001</a>; <a href="#B124">Suzzi et al., 2012</a>).</p>
<p class="mb15"><a href="#B118">Sipiczki (2011)</a> highlighted that <i>S</i>. <i>cerevisiae</i> wine strains are polyclonal and that the clones can differ significantly in oenological performance and genotype. The genomes of yeasts are subjected to duplications, deletions, and rearrangements that may cause the acquisition of new functions and gene specialization (<a href="#B30">Cubillos et al., 2011</a>). Some authors (<a href="#B53">Guijo et al., 1997</a>; <a href="#B87">Nadal et al., 1999</a>) highlighted that aneuploidy may be a method of yeast adaptation through the modification of the expression of some genes involved in this process (<a href="#B64">Legras et al., 2007</a>). In any case, aneuploids (<a href="#B57">Infante et al., 2003</a>; <a href="#B14">Bradbury et al., 2005</a>; <a href="#B64">Legras et al., 2007</a>; <a href="#B73">Lopandic et al., 2007</a>), triploids (<a href="#B31">Cummings and Fogel, 1978</a>; <a href="#B125">Takahashi, 1978</a>; <a href="#B126">Thornton, 1986</a>), polyploids (e.g., <a href="#B125">Takahashi, 1978</a>; <a href="#B4">Bakalinsky and Snow, 1990</a>; <a href="#B53">Guijo et al., 1997</a>; <a href="#B88">Naumov et al., 2000</a>, <a href="#B89">2002</a>) and rarely haploids (<a href="#B73">Lopandic et al., 2007</a>) may be present in the natural yeast biota of fermenting wine (<a href="#B118">Sipiczki, 2011</a>).</p>
<p class="mb15">Some studies have shown that genomic variability depends on telomeric recombination, which is important for adaptation to new environments and different metabolic sources and to overcome environmental stress, and on the insertion of transposable elements. Transposable elements comprise &#x0007E;3% of the total sequenced genome of <i>S</i>. <i>cerevisiae</i> S288c (<a href="#B18">Carreto et al., 2008</a>).</p>
<p class="mb0"><a href="#B18">Carreto et al. (2008)</a> showed that wine strains differed dramatically from the reference laboratory strain in Ty element composition, whereas clinical strains were similar to S288C in Ty element composition. Thus, it is likely that clinical strains and S288C had a common ancestor, and the differences found in wine strains may be due to the selective pressures that affect particular regions of the genome in response to adaptation to the environment. In particular, the variable genes were involved in metabolic functions related to cellular homeostasis or transport of different solutes such as ions, sugars, and metals. To better understand the population structure of wine <i>S</i>. <i>cerevisiae</i> strains, ecological studies using a polyphasic approach in order to define the biogeographical patterns have been carried out: a strict collaboration between phylogeneticists and ecologists and the development of new statistical tools provide a more comprehensive understanding of the factors controlling the <i>S</i>. <i>cerevisiae</i> biodiversity and biogeochemistry. The main molecular methods used for biogeographical studies are reported in Table <a href="#T3">3</a>.</p>
<div class="DottedLine"></div>
<div class="Imageheaders">TABLE 3</div>
<div class="FigureDesc">
<a href="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_m/fmicb-04-00166-t003.jpg" name="table3" target="_blank">
<img src="http://www.frontiersin.org/files/Articles/46996/fmicb-04-00166-r2/image_t/fmicb-04-00166-t003.gif" id="T3" alt="www.frontiersin.org" /></a>
<p><strong>TABLE 3. Molecular approaches used for <i>S</i>. <i>cerevisiae </i>biogeographical studies.</strong></p>
</div>
<div class="clear"></div>
<div class="DottedLine"></div>
<h3>CGH Array-Based Comparative Genomic Hybridization</h3>
<p class="mb0">Comparative genomic hybridization (CGH) is capable of detecting loss, gain and amplification of copy number at the chromosome level. Detection of amplifications is known to be sensitive down to less than 1 Mb. Therefore, one must take into consideration that although CGH is sensitive to specific types of copy number gains, its resolution for regional deletions is more limited. The use of array CGH overcomes this limitation, with improvements in resolution and dynamic range, in addition to the ability to directly map aberrations to the genome sequence and improved throughput (<a href="#B138">Weiss et al., 1999</a>). This approach has been recently applied to investigate the evolutionary importance of genome size in <i>S</i>. <i>cerevisiae</i> (<a href="#B38">Edwards-Ingram et al., 2004</a>; <a href="#B36">Dunn et al., 2005</a>, <a href="#B37">2012</a>; <a href="#B49">Gerstein et al., 2006</a>). <a href="#B37">Dunn et al. (2012)</a> used this technique to study copy number variations (CNVs) across subtelomeric regions, non-S288c genomic regions, retrotransposons, and the non-nuclear mtDNA and 2-mm plasmids of 83 <i>S</i>. <i>cerevisiae</i> strains isolated from different industrial and natural environments. The obtained clusters for the different types of features showed that most of the CNVs occurred either in subtelomeric regions or among the classes of transposable elements and that there were no commercial wine strains that appeared to be absolutely identical to each other. Thus, these CNVs did not produce any clear phylogeny, so it is likely that an active interchange of these regions occurred rather than separate lineages descending from isolated ancestors, suggesting that most of these strains are the result of interbreeding between industrial and wild strains.</p>
<h3>Genome Sequence and Functional Annotation</h3>
<p class="mb15">The genetic diversity of <i>Saccharomyces</i> strains can also be assessed using genome sequencing and functional genomic analysis of transcript profiles. These approaches are useful to aid in the understanding of speciation, life history variation, conditional fitness trade-offs and the long term maintenance of complex genomic variation (<a href="#B109">Scannell et al., 2011</a>). Genome sequencing provides the most complete understanding of the genomic structure of an organism and allows wide comparisons to be made between related species. <a href="#B109">Scannell et al. (2011)</a> improved the genome sequences of three species belonging to the <i>Saccharomyces sensu stricto</i> complex (<i>S</i>. <i>bayanus</i> var. <i>uvarum</i>, <i>S</i>. <i>kudriavzevii</i>, and <i>S</i>. <i>mikatae</i>) and compared them with the genomes of <i>S</i>. <i>cerevisiae</i> and <i>S</i>. <i>paradoxus</i>. They identified 5261 annotated protein coding orthologs across all of the studied species. Moreover, they found genes that had been lost in one or more lineages. Generally, the lost genes were derived from yeast genome duplications, suggesting that this phenomenon still influences yeasts and contributes to phenotypic differentiation. These authors also detected lineage-specific gains and found, in particular, two horizontal gene transfers from bacteria. These genes differentiated the analyzed species, indicating their involvement in speciation and adaptation. Other authors such as <a href="#B106">Rolland et al. (2009)</a> have also identified the presence of horizontal gene transfers from bacteria, confirming that this phenomenon plays important functional and evolutionary roles.</p>
<p class="mb15">To characterize the genomes of large numbers of individuals, microarray-based methods provide an alternative to DNA sequencing. This method allows the identification of conserved and non-conserved regions across microbial populations. The use of tiling arrays followed by analysis of the DNA region via polymerase chain reaction (PCR) is useful to determine whether the absence of hybridization is due to deletion of a chromosomal region or due to areas of large sequence polymorphism (LSP; <a href="#B86">Muller and McCusker, 2011</a>). <a href="#B86">Muller and McCusker (2011)</a> characterized the genome-wide distribution of LSPs in 88 <i>S</i>. <i>cerevisiae </i>strains of diverse geographical origins and source substrates using high-density tiling arrays. They showed that LSPs occurred in the subtelomeric regions of chromosomes, where they did not disrupt essential gene expression. Moreover, this study revealed the presence of introgressions. In particular, clinical strains contained <i>S</i>. <i>paradoxus</i> DNA fragments. In another study, <a href="#B84">Muller and McCusker (2009a)</a> developed a multi-species-based taxonomic microarray consisting of features targeted to multiple orthologous genes from <i>S. cerevisiae, S. paradoxus, S. mikatae, S. bayanus, S. kudriavzevii, N. castellii, L. kluyveri</i>, and the closely related <i>Candida glabrata</i>. In particular, they studied 183 supposed <i>S. cerevisiae </i>isolates of diverse ecological and geographical origins. They again confirmed the existence of introgressions in wine strains, identifying four hybrids, one between <i>S. cerevisiae </i>and <i>S. bayanus </i>and three between <i>S. cerevisiae </i>and <i>S. kudriavzevii</i>. In addition, this approach allowed the detection of multiple introgressed <i>S. paradoxus </i>DNA fragments in the genomes of three different <i>S. cerevisiae</i> isolates.</p>
<p class="mb0">Other researchers studied comparative transcript profiling to define the relationships among strains (<a href="#B61">Kvitek et al., 2008</a>; <a href="#B20">Cavalieri, 2009</a>; <a href="#B16">Bullard et al., 2010</a>). Conservation of a transcriptional response indicates functional relatedness of the organisms under investigation (<a href="#B67">Lelandais et al., 2011</a>). In fact, genome rearrangements can modify gene expression and alter phenotypes. <a href="#B61">Kvitek et al. (2008)</a> measured whole-genome expression in 52 strains collected from different niches (industrial processes and human illnesses) in the presence of different stress conditions. Wine strains were able to grow in the majority of the tested conditions; for example, copper resistance was predominant in wine strains, suggesting that the use of copper in the vineyard strongly selected against strains that were copper sensitive (<a href="#B61">Kvitek et al., 2008</a>). This evidence confirmed that the process of fermentation imposes a strong selective pressure and therefore is a powerful evolutionary force in the generation of diversity (<a href="#B61">Kvitek et al., 2008</a>; <a href="#B30">Cubillos et al., 2011</a>; <a href="#B118">Sipiczki, 2011</a>).</p>
<h3>Pulsed-Field Gel Electrophoresis</h3>
<p class="mb15">Wine strains generally have a large diversity in the number and size of chromosomes that can be observed by pulsed-field gel electrophoresis (PFGE) analysis, which separates chromosome-sized DNA molecules. This method was first described by <a href="#B116">Schwartz and Cantor (1984)</a> and is still one of the most powerful tools to investigate the biogeography and speciation of this yeast in nature. Analysis of the chromosomes of wine yeast strains by PFGE demonstrated the presence of chromosome-length polymorphisms, which are derived from chromosomal rearrangements such as translocations and deletions (<a href="#B19">Carro et al., 2003</a>). <a href="#B19">Carro et al. (2003)</a> suggested that the subtelomeric plasticity of chromosome I, which contains several membrane-associated genes, may induce rapid adaptive changes of the yeast strains in response to specific environmental cues (substrates). The reciprocal translocation between chromosomes VIII and XVI generated the SSU1-R allele, which confers sulfite resistance to yeast cells and was described as the first case of adaptive evolution, likely occurring as a consequence of the use of sulfites as a preservative in wine production (<a href="#B52">Goto-Yamamoto et al., 1998</a>; <a href="#B95">Perez-Ortin et al., 2002</a>). Many authors have showed that karyotyping is more discriminative than other approaches for yeast typing because it is able to highlight polymorphisms in electrophoretic chromosomal profiles in natural <i>S</i>. <i>cerevisiae</i> populations from almost all wine-growing regions of the world (<a href="#B59">Johnston and Mortimer, 1986</a>; <a href="#B131">Vezinhet et al., 1990</a>, <a href="#B132">1992</a>; <a href="#B8">Bidenne et al., 1992</a>; <a href="#B47">Frezier and Dubourdieu, 1992</a>; <a href="#B15">Briones et al., 1996</a>; <a href="#B39">Egli et al., 1998</a>; <a href="#B96">Povhe et al., 2001</a>; <a href="#B115">Schuller et al., 2004</a>; <a href="#B119">Sipiczki et al., 2004</a>; <a href="#B2">Antunovics et al., 2005</a>; <a href="#B1">Aa et al., 2006</a>; <a href="#B134">Wang et al., 2012</a>).</p>
<p class="mb0">This approach showed that strains isolated from the same fermentation generally differ in chromosomal length (<a href="#B39">Egli et al., 1998</a>; <a href="#B81">Mesa et al., 1999</a>; <a href="#B120">Sipiczki et al., 2001</a>, <a href="#B119">2004</a>; <a href="#B2">Antunovics et al., 2005</a>), indicating that clones with different sets of chromosomes propagate at the same time and in succession during fermentation (<a href="#B118">Sipiczki, 2011</a>). <a href="#B134">Wang et al. (2012)</a> applied this technique to type <i>S</i>. <i>cerevisiae</i> strains with different ecological and geographical origins to better understand the ecology of <i>S. cerevisiae</i>. The obtained results showed that a wide divergence of populations of wild <i>S. cerevisiae</i> exist and that this divergence is only marginally affected by human activity. <a href="#B36">Dunn et al. (2005)</a> revealed the existence of a set of deleted or amplified genes common to wine and other industrial yeasts, and certain genes have been identified as a possible wine yeast signature, particularly genes encoding membrane transporters.</p>
<h3>mtDNA-RFLP</h3>
<p class="mb15"><i>Saccharomyces cerevisiae</i> mtDNA is characterized by an elevated mutation rate. In particular, base-substitution mutations and length polymorphisms can be highlighted by restriction fingerprinting of mtDNA using endonucleases with different target sites (e.g., <i>Dde</i>I, <i>Hinf</i>I, <i>Alu</i>I, and <i>Rsa</i>I). The reliability and discrimination power of this fingerprinting technique are similar to those of PFGE.</p>
<p class="mb0">The use of mtDNA-restriction fragment length polymorphism (RFLP) revealed a wide range of polymorphisms in mitochondrial genomes and mitochondrial genes (<a href="#B131">Vezinhet et al., 1990</a>; <a href="#B99">Querol et al., 1994</a>; <a href="#B130">Versavaud et al., 1995</a>; <a href="#B75">Lopez et al., 2003</a>). This technique was used together with PFGE by <a href="#B132">Vezinhet et al. (1992)</a> to study the evolution of <i>S</i>. <i>cerevisiae</i> strains isolated from different wine regions over 6 years. The study demonstrated that some strains were widely distributed in the studied areas and present over several years, indicating that they are endemic to that region. More recently, <a href="#B33">Di Maio et al. (2012)</a> used this method to investigate the biodiversity of wine yeast populations isolated over several years from Sicilian wineries where commercial yeast strains have never been used. mtDNA-RFLP allowed the differentiation of 209 of 918 yeast strains. <a href="#B111">Schuller et al. (2005)</a> performed a large-scale biogeographical survey on the genetic diversity of <i>S. cerevisiae</i> strains isolated from spontaneous fermentations and identified 297 different genetic patterns among 1620 strains isolated from 54 small-scale fermentations of grapes from three vineyards located in the Vinho Verde region (Portugal) during a 3 year period. Almost all of the obtained patterns were unique, showing the large biodiversity of <i>S</i>. <i>cerevisiae</i> in that region.<i></i></p>
<h3>RAPD-PCR</h3>
<p class="mb0">This technique is based on the use of a single short primer (8&#x02013;12 nucleotides) that amplifies &#x0201C;anonymous&#x0201D; DNA sequences and represents a powerful typing method for many yeast and bacterial species (<a href="#B100">Quesada and Cenis, 1995</a>; <a href="#B76">Martinez et al., 2007</a>; <a href="#B127">Tofalo et al., 2007</a>). In fact, the annealing of the primer at several points allows the user to obtain a complex banding pattern that is specific for each strain (<a href="#B58">Ivey and Phister, 2011</a>). This method was used by <a href="#B21">Cavalieri et al. (1998)</a> to differentiate 166 <i>S</i>. <i>cerevisiae </i>strains isolated from Tuscany and Sicily, two Italian regions. In this case, random amplified polymorphic DNA (RAPD)-PCR allowed the recognition of 16 patterns, and only 10 were strain specific. <a href="#B127">Tofalo et al. (2007)</a> used this approach to recognize genetically different <i>S. cerevisiae </i>strains, which were clustered in subgroups related to the four different wine-producing areas of the Apulia region (Italy). The obtained results showed that the genetic differences reflect the phenotypic biodiversity.</p>
<h3>Microsatellite Analyses</h3>
<p class="mb0">Microsatellites, also known as simple sequence repeats (SSRs) or short tandem repeats (STRs), are repeating sequences of 1&#x02013;6 base pairs of DNA that are characterized by a high level of polymorphism. They occur within many open reading frames but are even more frequent in non-coding regulatory regions. In <i>S</i>. <i>cerevisiae</i>, microsatellites have been described as abundant and highly polymorphic in length (<a href="#B104">Richards et al., 2009</a>), and for this reason, they are used as a reproducible and portable typing method (<a href="#B48">Gallego et al., 1998</a>; <a href="#B55">Hennequin et al., 2001</a>; <a href="#B115">Schuller et al., 2004</a>; <a href="#B14">Bradbury et al., 2005</a>; <a href="#B65">Legras et al., 2005</a>). Recently, an increasing number of microsatellites have been described for <i>S</i>. <i>cerevisiae</i>, with the aim of identifying the most polymorphic loci with a high allelic diversity that can be used for both strain identification and the establishment of strain geographical or technological origin. Several studies used this approach to type <i>S. cerevisiae</i> strains of different geographical origins (<a href="#B91">Ness et al., 1993</a>; <a href="#B130">Versavaud et al., 1995</a>; <a href="#B48">Gallego et al., 1998</a>; <a href="#B55">Hennequin et al., 2001</a>; <a href="#B14">Bradbury et al., 2005</a>; <a href="#B65">Legras et al., 2005</a>; <a href="#B111">Schuller et al., 2005</a>, <a href="#B114">2007</a>; <a href="#B85">Muller and McCusker, 2009b</a>). For example, <a href="#B113">Schuller and Casal (2007)</a> analyzed six polymorphic microsatellite loci in 361 strains isolated from the Vinho Verde region in Portugal during the 2001&#x02013;2003 harvest seasons. Fifty-two new alleles were identified in addition to the 41 alleles previously described (ScAAT1&#x02013;ScAAT6). Recently, a database of 246 genotypes has been compiled that includes 78 commercial strains of wine yeast, a range of yeast isolates from New Zealand wineries, and natural yeast strains from around the world, including 35 that were recently sequenced (<a href="#B71">Liti et al., 2009</a>). Regardless of the technique chosen, a combination of different techniques is necessary to obtain unambiguous results. For example, <a href="#B115">Schuller et al. (2004)</a> showed that genotypes with the same microsatellite pattern (using six loci) can have different karyotypes. This group also identified a large number of variants of a commercial wine strain that had escaped to adjacent vineyards (<a href="#B114">Schuller et al., 2007</a>). <a href="#B3">Ayoub et al. (2006)</a> also found that genotypes that could not be resolved by microsatellite profiles were sometimes discriminated by interdelta PCR or by sequence analysis. In <i>S. cerevisiae</i> 84 minisatellites have been reported, but recently four tandem repeated motif of 135 bp or larger called megasatellites have been described (<a href="#B105">Rolland et al., 2010</a>). They are found in paralogous <i>FLO1</i>, <i>FLO5</i>, <i>FLO9</i>, and <i>NUM</i>1 genes. These motifs could be as targets to measure evolutionary relationships at intra- and intergenic levels (<a href="#B105">Rolland et al., 2010</a>).</p>
<h3>&#x003B4; Sequences</h3>
<p class="mb0">The &#x003B4; sequences are flanking sequences (300 bp) frequently associated with the Ty1 and Ty2 transposons that are dispersed throughout the genome and are particularly common in terminal chromosomal regions (<a href="#B46">Franco-Duarte et al., 2011</a>). They are also found as single elements. The number (from 35 to 55) and the location of these elements are variable among species, so the &#x003B4; sequences represent useful genetic markers for the identification of polymorphisms. Amplification of interdelta regions between neighboring d sequences generates strain-specific banding patterns. This method is suitable for the characterization of high numbers of strains because it is easy to perform, cheap and rapid. Recently, alternative primers (d12 and d21) that bind close to the initially described binding sites for primers d1 and d2 (<a href="#B63">Legras and Karst, 2003</a>) were designed to improve this method (<a href="#B91">Ness et al., 1993</a>). The combination of these primers (d12/&#x003B4;21 or d12/d2) increased the discriminatory power of the method (<a href="#B65">Legras et al., 2005</a>). In particular, the use of primer pairs d12/d2 showed the same discriminatory power as other methods, such as mtDNA, RFLP, microsatellite analysis, and karyotyping, for strain typing (<a href="#B115">Schuller et al., 2004</a>). <a href="#B112">Schuller et al. (2012)</a> used this approach to study the intraspecific genetic diversity of vineyard-associated <i>S</i>. <i>cerevisiae</i> strains. In particular, grapes were harvested from 16 vineyards over 2 years. A strict correlation between genotype and grape variety was found.</p>
<h3>Multilocus Sequence Typing</h3>
<p class="mb15">Another technique used for <i>S</i>. <i>cerevisiae</i> strain typing is multilocus sequence typing (MLST), which was recently shown to be a powerful technique for typing microorganisms (<a href="#B1">Aa et al., 2006</a>; <a href="#B3">Ayoub et al., 2006</a>). Strains are characterized using the DNA sequences of internal fragments of multiple housekeeping genes where variation accumulates relatively slowly and tends to be selectively neutral. It is highly reliable and highly discriminatory at the strain level, and because it is based on nucleotide sequencing, the results are easily comparable between laboratories.</p>
<p class="mb15">Recently, this technique was applied to study <i>S</i>. <i>cerevisiae</i> population structure and evolution (<a href="#B42">Fay and Benavides, 2005</a>; <a href="#B1">Aa et al., 2006</a>). <a href="#B3">Ayoub et al. (2006)</a> tested a set of seven loci of 84 <i>S</i>. <i>cerevisiae</i> strains of different origins: 65 strains were isolated from traditional wineries in Lebanon, and the others were commercial wine strains and Asian isolates. MLST profiling allowed the differentiation of the Asian group of strains from the Lebanese and European commercial strains that appear closely related, suggesting the introduction of genetic material from Asian strains into Lebanon.</p>
<p class="mb0"><a href="#B133">Vigentini et al. (2009)</a> studied the genetic biodiversity of an <i>S. cerevisiae</i> collection including 33 commercial strains, 14 wine isolates, and three laboratory strains by screening for single-nucleotide polymorphisms (SNPs) in loci on genes involved in wine production. In particular, they focused on the identification of SNPs as new genetic markers. Several studies report the efficacy of this analysis for studying the evolution of a microbial population (<a href="#B7">Ben-Ari et al., 2005</a>; <a href="#B1">Aa et al., 2006</a>). The obtained results showed that the collection was characterized by a low polymorphism rate and degree of heterozygosity and that the gene coding for the trehalose-6-phosphate synthase enzyme, which is involved in ethanol resistance, could be used as a molecular target. In fact, this gene showed a sequence diversity of 1.42% with seven different nucleotide substitutions.</p>
<a id="h5" name="h5"></a><h2>Conclusion</h2>
<p class="mb0">Biogeographical studies revealed that <i>S</i>. <i>cerevisiae</i> species consists of both &#x0201C;domesticated&#x0201D; and &#x0201C;wild&#x0201D; populations which are phylogenetically distinct. These populations probably derives from the whole-genome duplication of a common ancestor strain. In particular, for <i>S</i>. <i>cerevisiae</i> wine yeast a clear geographical origin was established at regional and global scales suggesting that the different strains evolved independently for long time. They modified the dosage of some genes important for the persistence in specific ecological niches which represent a reservoir of natural yeasts. Comparative genomics studies highlighted that <i>S</i>. <i>cerevisiae</i> wine strains differ not only for their origin but also for genetic transfers from other yeasts (<i>Saccharomyces</i> and non-<i>Saccharomyces</i>) and bacteria. The main difference have been found in genes encoding cell wall proteins and associated with aminoacid uptake which are important for the production of sensorially volatile aroma compounds. So, the genomic tools are crucial to better understand genetic and molecular basis of yeast evolution and the &#x0201C;art&#x0201D; of wine making, so characterization of other yeast environmental isolates will be useful to develop tailor strains to meet consumer demand.</p>
<a id="h6" name="h6"></a><h2>Conflict of Interest Statement</h2>
<p class="mb0">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p>
<a id="h7" name="h7"></a><h2>Acknowledgments</h2>
<p class="mb0">This research has received funding from Cassa di Risparmio di Teramo (Oenological microbiota: selection to identify the wine character and to improve the competitiveness of Montepulciano d&#x02019;Abruzzo wineries).</p>
<a id="h8" name="h8"></a><h2>References</h2>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B1" id="B1"></a>Aa, E., Townsend, J. P., Adams, R. I., Nielsen, K. M., and Taylor, J. W. (2006). Population structure and gene evolution in <i>Saccharomyces cerevisiae</i>. <i>FEMS Yeast Res. </i>6, 702&#x02013;715. doi: 10.1111/j.1567-1364.2006.00059.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16879422" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16879422" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2006.00059.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B2" id="B2"></a>Antunovics, Z., Irinyi, L., and Sipiczki, M. (2005). Combined application of methods to taxonomic identification of <i>Saccharomyces</i> strains in fermenting botrytized grape must. <i>J. Appl. Microbiol. </i>98, 971&#x02013;979. doi: 10.1111/j.1365-2672.2005.02543.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15752344" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15752344" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-2672.2005.02543.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B3" id="B3"></a>Ayoub, M. J., Legras, J. L., Saliba, R., and Gaillardin, C. (2006). Application of multi locus sequence typing to the analysis of the biodiversity of indigenous <i>Saccharomyces cerevisiae</i> wine yeasts from Lebanon. <i>J. Appl. Microbiol. </i>100, 699&#x02013;711. doi: 10.1111/j.1365-2672.2006.02817.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16553725" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16553725" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-2672.2006.02817.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B4" id="B4"></a>Bakalinsky, A. T., and Snow, R. (1990). The chromosomal constitution of wine strains of <i>Saccharomyces cerevisiae</i>. <i>Yeast </i>6, 367&#x02013;382. doi: 10.1002/yea.320060503</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=2220073" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=2220073" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1002/yea.320060503" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B5" id="B5"></a>Barata, A., Gonzalez, S., Malfeito-Ferreira, M., Querol, A., and Loureiro, V. (2008). Sourrot-damaged grapes are sources of wine spoilage yeasts. <i>FEMS Yeast Res. </i>8, 1008&#x02013;1017. doi: 10.1111/j.1567-1364.2008.00399.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18554306" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18554306" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2008.00399.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B6" id="B6"></a>Barata, A., Malfeito-Ferreira, M., and Loureiro, V. (2012). The microbial ecology of wine grape berries. <i>Int. J. Food Microbiol. </i>153, 243&#x02013;259. doi: 10.1016/j.ijfoodmicro.2011.11.025</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22189021" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22189021" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2011.11.025" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B7" id="B7"></a>Ben-Ari, G., Zenvirth, D., Sherman, A., Simchen, G., Lavi, U., and Hillel, J. (2005). Application of SNPs for assessing biodiversity and phylogeny among yeast strains. <i>Heredity </i>95, 493&#x02013;501. doi: 10.1038/sj.hdy.6800759</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16175193" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16175193" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/sj.hdy.6800759" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B8" id="B8"></a>Bidenne, C., Blondin, B., Dequin, S., and Vezinhet, F. (1992). Analysis of the chromosomal DNA polymorphism of wine strains of <i>Saccharomyces</i> cerevisiae. <i>Curr. Genet. </i>22, 1&#x02013;7. doi: 10.1007/BF00351734</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=1611665" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=1611665" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/BF00351734" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B9" id="B9"></a>Blondin, B., Dequin, S., Amparo, Q., and Legras, J. L. (2011). &#x0201C;Genome of <i>Saccharomyces cerevisiae</i> and related yeasts,&#x0201D; in <i>Biology of Microorganisms on Grapes, in Must and in Wine</i>, eds H. K&#x000F6;nig, G. Unden, and J. Fr&#x000F6;hlich (Berlin: Springer-Verlag), 361&#x02013;378.</p>
</div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B10" id="B10"></a>Borneman, A. R., Desany, B. A., Riches, D., Affourtit, J. P., Forgan, A. H., Pretorius, I. S., et al. (2012). The genome sequence of the wine yeast VIN7 reveals an allotetraploid hybrid genome with <i>Saccharomyces cerevisiae</i> and <i>Saccharomyces kudriavzevii</i> origins. <i>FEMS Yeast Res. </i>12, 88&#x02013;96. doi: 10.1111/j.1567-1364.2011.00773.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22136070" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22136070" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2011.00773.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B11" id="B11"></a>Borneman, A. R., Desany, B. A., Riches, D., Affourtit, J. P., Morgan, A. H., Pretorius, I. S., et al. (2011). Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of <i>Saccharomyces cerevisiae</i>. <i>PLoS Genet. </i>7:e1001287. doi: 10.1371/journal.pgen.1001287</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21304888" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21304888" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pgen.1001287" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B12" id="B12"></a>Borneman A. R., Forgan A. H., Pretorius, I. S., and Chambers, P. J. (2008). Comparative genome analysis of a <i>Saccharomyces cerevisiae</i> wine strain. <i>FEMS Yeast Res. </i>8, 1185&#x02013;1195. doi: 10.1111/j.1567-1364.2008.00434.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18778279" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18778279" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2008.00434.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B13" id="B13"></a>Borneman, A. R., Pretorius, I. S., and Chambers, P. J. (2013). Comparative genomics: a revolutionary tool for wine yeast strain development. <i>Curr. Opin. Biotechnol. </i>24, 192&#x02013;199. doi: 10.1016/j.copbio.2012.08.006</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22947601" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22947601" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.copbio.2012.08.006" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B14" id="B14"></a>Bradbury, J. E., Richards, K. D., Niederer, H. A., Lee S. A., Dunbar, P. R., and Gardner, R. C. (2005). A homozygous diploid subset of commercial wine yeast strains. <i>Antonie van Leeuwenhoek </i>89, 27&#x02013;37. doi: 10.1007/s10482-005-9006-1</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16328862" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16328862" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s10482-005-9006-1" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B15" id="B15"></a>Briones, A. I., Ubeda, J., and Grando, M. S. (1996). Differentiation of <i>Saccharomyces cerevisiae</i> strains isolated from fermenting musts according to their karyotype pattern. <i>Int. J. Food Microbiol. </i>28, 369&#x02013;377. doi: 10.1016/0168-1605(95)00008-9</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=8652345" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=8652345" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/0168-1605(95)00008-9" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B16" id="B16"></a>Bullard, J. H., Mostovoy, Y., Dudoit, S., and Brem, R. B. (2010). Polygenic and directional regulatory evolution across pathways in <i>Saccharomyces</i>. <i>Proc. Natl. Acad. Sci. U.S.A. </i>107, 5058&#x02013;5063. doi: 10.1073/pnas.0912959107</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=20194736" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=20194736" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1073/pnas.0912959107" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B17" id="B17"></a>Camarasa, C., Sanchez, I., Brial, P., Bigey, F., and Dequin, S. (2011). Phenotypic landscape of <i>Saccharomyces cerevisiae</i> during wine fermentation: evidence for origin-dependent metabolic traits. <i>PLoS ONE </i>6:e25147. doi: 10.1371/journal.pone.0025147</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21949874" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21949874" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pone.0025147" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B18" id="B18"></a>Carreto, L., Eiriz, M. F., Gomes, A. C., Pereira, P. M., Schuller, D., and Santos M. A. S. (2008). Comparative genomics of wild type yeast strains unveils important genome diversity. <i>BMC Genomics </i>9:524. doi: 10.1186/1471-2164-9-524</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18983662" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18983662" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1186/1471-2164-9-524" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B19" id="B19"></a>Carro, D., Garcia-Martinez, J., Prez-Ortin, J. E., and Pina, B. (2003). Structural characterization of chromosome I size variant from a natural yeast strain. <i>Yeast </i>20, 171&#x02013;183. doi: 10.1002/yea.955</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=12518320" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=12518320" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1002/yea.955" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B20" id="B20"></a>Cavalieri, D. (2009). Evolution of transcriptional regulatory networks in yeast populations. <i>Wiley Interdiscip</i>.<i> Rev.</i> <i>Syst. Biol. Med.</i> 2, 324&#x02013;335. doi: 10.1002/wsbm.68</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=20836032" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=20836032" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1002/wsbm.68" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B21" id="B21"></a>Cavalieri, D., Barberio, C., Casalone, E., Pinzauti, F., Sebastian, F., Mortimer, R., et al. (1998). Genetic and molecular diversity in <i>Saccharomyces cerevisiae</i> natural populations. <i>Food Technol. Biotechnol. </i>36, 45&#x02013;50. doi: 10.1534/genetics.106.062745</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1534/genetics.106.062745" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B22" id="B22"></a>Cavalieri, D., McGovern, P. E., Hartl, D. L., Mortimer, R., and Polsinelli, M. (2003). Evidence for <i>S. cerevisiae</i> fermentation in ancient wine.<i> J. Mol. Evol. </i>57, S226&#x02013;S232. doi: 10.1007/s00239-003-0031-2</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15008419" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15008419" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s00239-003-0031-2" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B23" id="B23"></a>Chavan, P., Mane, S., Kulkarni, G., Shaikh, S., Ghormade, V., Nerkar, D. P., et al. (2009). Natural yeast flora of different varieties of grapes used for wine making in India. <i>Food Microbiol. </i>26, 801&#x02013;808. doi: 10.1016/j.fm.2009.05.005</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19835764" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19835764" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.fm.2009.05.005" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B24" id="B24"></a>Christen, R. (2008). Global sequencing: a review of current molecular data and new methods available to assess microbial diversity. <i>Microbes Environ. </i>23, 253&#x02013;268. doi: 10.1264/jsme2.ME08525</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21558717" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21558717" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1264/jsme2.ME08525" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B25" id="B25"></a>Ciani, M., Mannazzu, I., Marinangeli, P., Clementi, F., and Martini, A. (2004). Contribution of winery-resident <i>Saccharomyces cerevisiae</i> strains to spontaneous grape must fermentation. <i>Antonie van Leeuwenhoek </i>85, 159&#x02013;164. doi: 10.1023/B:ANTO.0000020284.05802.d7</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15031657" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15031657" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1023/B:ANTO.0000020284.05802.d7" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B26" id="B26"></a>Codon, A. C., Benitez, T., and Korhola, M. (1998). Chromosomal polymorphism and adaptation to specific industrial environments of <i>Saccharomyces</i> strains. <i>Appl. Microbiol. Biotechnol. </i>49, 154&#x02013;163. doi: 10.1007/s002530051152</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=9534255" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=9534255" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s002530051152" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B27" id="B27"></a>Combina, M., Mercado, L., Borgo, P., Elia, A., Jofr&#x000E9;, V., Ganga, A., et al. (2005). Yeasts associated to Malbec grape berries from Mendoza, Argentina. <i>J. Appl. Microbiol. </i>98, 1055&#x02013;1061. doi: 10.1111/j.1365-2672.2005.02540.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15836474" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15836474" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-2672.2005.02540.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B28" id="B28"></a>Constanti, M., Poblet, M., Arola, L., Mas, A., and Guillamon, J. M. (1997). Analysis of yeast populations during alcoholic fermentation in a newly established winery. <i>Am. J. Enol. Vitic. </i>48, 339&#x02013;344. doi: 10.1023/A:1012027718701</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1023/A:1012027718701" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B29" id="B29"></a>Cordero-Bueso, G., Arroyo, T., Serrano, A., Tello, J., Aporta, I., V&#x000E9;lez, M. D., et al. (2011). Influence of the farming system and vine variety on yeast communities associated with grape berries. <i>Int. J. Food Microbiol. </i>145, 132&#x02013;139. doi: 10.1016/j.ijfoodmicro.2010.11.040</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21185102" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21185102" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2010.11.040" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B30" id="B30"></a>Cubillos, F. A., Billi, E., Zorgo, E., Parts, L., Fargier, P., Omholt, S., et al. (2011). Assessing the complex architecture of polygenic traits in diverged yeast populations. <i>Mol. Ecol. </i>20, 1401&#x02013;1413. doi: 10.1111/j.1365-294X.2011.05005.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21261765" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21261765" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-294X.2011.05005.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B31" id="B31"></a>Cummings, J., and Fogel, S. (1978). Genetic homology of wine yeasts with <i>Saccharomyces cerevisiae</i>. <i>J. Inst. Brew. </i>84, 267&#x02013;270. doi: 10.1002/j.2050-0416.1978.tb03885.x</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1002/j.2050-0416.1978.tb03885.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B32" id="B32"></a>Dequin, S. (2001). The potential of genetic engineering for improving brewing, wine-making and baking yeasts. <i>Appl. Microbiol. Biotechnol. </i>56, 577&#x02013;588. doi: 10.1007/s002530100700</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=11601604" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=11601604" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s002530100700" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B33" id="B33"></a>Di Maio, S., Polizzotto, G., Di Gangi, E., Foresta, G., Genna, G., Verzera, A., et al. (2012). Biodiversity of indigenous <i>Saccharomyces</i> populations from old wineries of South-Eastern Sicily (Italy): preservation and economic potential. <i>PLoS ONE </i>7:e30428. doi: 10.1371/journal.pone.0030428</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22393353" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22393353" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pone.0030428" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B34" id="B34"></a>Doniger, S. W., Kim, H. S., Swain, D., Corcuera, D., Williams, M., Yang, S. P., et al. (2008). A catalog of neutral and deleterious polymorphism in yeast. <i>PLoS Genet.</i> 4:e1000183. doi: 10.1371/journal.pgen.1000183</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18769710" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18769710" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pgen.1000183" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B35" id="B35"></a>Dujon, B. (2006). Yeasts illustrate the molecular mechanisms of eukaryotic genome evolution. <i>Trends Genet. </i>22, 375&#x02013;387. doi: 10.1016/j.tig.2006.05.007</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16730849" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16730849" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.tig.2006.05.007" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B36" id="B36"></a>Dunn, B., Levine, R. P., and Sherlock, G. (2005). Microarray karyotyping of commercial wine yeast strains reveals shared, as well as unique, genomic signatures. <i>BMC Genomics </i>6:53. doi: 10.1186/1471-2164-6-53</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15833139" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15833139" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1186/1471-2164-6-53" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B37" id="B37"></a>Dunn, B., Richter, C., Kvitek, D. J., Pugh, T., and Sherlock, G. (2012). Analysis of the <i>Saccharomyces cerevisiae</i> pan-genome reveals a pool of copy number variants distributed in diverse yeast strains from differing industrial environments. <i>Genome Res. </i>22, 908&#x02013;924. doi: 10.1101/gr.130310.111</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22369888" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22369888" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1101/gr.130310.111" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B38" id="B38"></a>Edwards-Ingram, L. C., Gent, M. E., Hoyle, D. C., Hayes, A., Stateva, L. I., and Oliver, S. (2004). Comparative genomic hybridization provides new insights into the molecular taxonomy of the <i>Saccharomyces sensu stricto</i> complex. <i>Genome Res. </i>14, 1043&#x02013;1051. doi: 10.1101/gr.2114704</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15173111" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15173111" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1101/gr.2114704" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B39" id="B39"></a>Egli, C. M., Edinger, W. D., Mitrakul, C. M., and Henick-Kling, T. (1998). Dynamics of indigenous and inoculated yeast populations and their effect on the sensory character of riesling and Chardonnay. <i>J. Appl. Microbiol. </i>85, 779&#x02013;789. doi: 10.1046/j.1365-2672.1998.00521.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=9830113" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=9830113" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1046/j.1365-2672.1998.00521.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B40" id="B40"></a>Ehrenreich, I. M., Torabi, N., Jia, Y., Kent, J., Martis, S., Shapiro, J. A., et al. (2010). Dissection of genetically complex traits with extremely large pools of yeast segregants. <i>Nature </i>464, 1039&#x02013;1042. doi: 10.1038/nature08923</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=20393561" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=20393561" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/nature08923" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B41" id="B41"></a>Erny, C., Raoult, P., Alias, A., Butterlin, G., Delobel, P., Matei-Radoi, F., et al. (2012). Ecological success of a group of <i>Saccharomyces cerevisiae</i>/<i>Saccharomyces</i> <i>kudriavzevii</i> hybrids in the Northern European wine-making environment. <i>Appl. Environ. Microbiol. </i>78, 3256&#x02013;3265. doi: 10.1128/AEM.06752-11</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22344648" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22344648" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1128/AEM.06752-11" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B42" id="B42"></a>Fay, J. C., and Benavides, J. A. (2005). Evidence for domesticated and wild populations of <i>Saccharomyces cerevisiae</i>. <i>PLoS Genet. </i>1:e5. doi: 10.1371/journal.pgen.0010005</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16103919" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16103919" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pgen.0010005" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B43" id="B43"></a>Fidalgo, M., Barrales, R. R., Ibeas, J. I., and Jimenez, J. (2006). Adaptive evolution by mutations in the FLO11 gene. <i>Proc. Natl. Acad. Sci. U.S.A. </i>103, 11228&#x02013;11233. doi: 10.1073/pnas.0601713103</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16844788" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16844788" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1073/pnas.0601713103" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B44" id="B44"></a>Fleet, G. H. (2003). Yeast interactions and wine flavour. <i>Int. J. Food Microbiol. </i>86, 11&#x02013;22. doi: 10.1016/S0168-1605(03)00245-9</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=12892919" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=12892919" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/S0168-1605(03)00245-9" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B45" id="B45"></a>Francesca, N., Canale, D. E., Settanni, L., and Moschetti, G. (2012). Dissemination of wine-related yeasts by migratory birds. <i>Environ. Microbiol. Rep. </i>4, 105&#x02013;112. doi: 10.1111/j.1758-2229.2011.00310.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=23757236" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=23757236" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1758-2229.2011.00310.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B46" id="B46"></a>Franco-Duarte, R., Mendes. I., Gomes. A. C., Santos, M. A. S., de Sousa, B., and Schuller D. (2011). Genotyping of <i>Saccharomyces cerevisiae</i> strains by interdelta sequence typing using automated microfluidics. <i>Electrophoresis </i>32, 1447&#x02013;1455. doi: 10.1002/elps.201000640</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21630290" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21630290" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1002/elps.201000640" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B47" id="B47"></a>Frezier, V., and Dubourdieu, D. (1992). Ecology of yeast strains of <i>Saccharomyces cerevisiae</i> during spontaneous fermentation in Bordeaux winery. <i>Am. J. Enol. Vitic. </i>43, 375&#x02013;380. doi: 10.2478/s11696-012-0198-3</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.2478/s11696-012-0198-3" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B48" id="B48"></a>Gallego, F. J., Perez, M. A., Martinez, I., and Hidalgo, P. (1998). Microsatellites obtained from database sequences are useful to characterize <i>Saccharomyces cerevisiae</i> strains. <i>Am. J. Enol. Vitic. </i>49, 350&#x02013;351. doi: 10.1128/AEM.01729-07</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1128/AEM.01729-07" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B49" id="B49"></a>Gerstein, A. C., Chun, H.-J. E., Grant, A., and Otto, S. P. (2006). Genomic convergence toward diploidy in <i>Saccharomyces cerevisiae</i>. <i>PLoS Genet. </i>2:e145. doi: 10.1371/journal.pgen.0020145</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17002497" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17002497" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pgen.0020145" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B50" id="B50"></a>Goffeau, A., Barrell, B. G., Bussey, H., Davis, R. W., and Dujon, B. (1996). Life with 6000 genes. <i>Science </i>274, 563&#x02013;567. doi: 10.1126/science.274.5287.546</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=8849441" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=8849441" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1126/science.274.5287.546" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B51" id="B51"></a>Gonzales, S. S., Barrio, E., Gafner, J., and Querol, A. (2006). Natural hybrids from <i>Saccharomyces cerevisiae Saccharomyces bayanus</i> and <i>Saccharomyces kudriavzevii</i> in wine fermentations. <i>FEMS Yeast Res. </i>6, 1221&#x02013;1234. doi: 10.1111/j.1567-1364.2006.00126.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17156019" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17156019" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2006.00126.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B52" id="B52"></a>Goto-Yamamoto, N., Kitano, K., and Shiki, K. (1998). SSU1-R, a sulphite resistance gene of wine yeast, is an allele of SSU1 with a different upstream sequence. <i>J. Ferm. Bioeng. </i>86, 427&#x02013;433. doi: 10.1016/S0922-338X(98)80146-3</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1016/S0922-338X(98)80146-3" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B53" id="B53"></a>Guijo, S., Mauricio, J. C., Salmon, J. M., and Ortega, J. M. (1997). Determination of the relative ploidy in different <i>Saccharomyces cerevisiae</i> strains used for fermentation and &#x02018;flor&#x02019; film ageing of dry sherry-type wines. <i>Yeast </i>13, 101&#x02013;117. doi: 10.1002/(SICI)1097-0061(199702)13:2</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1002/(SICI)1097-0061(199702)13:2&#x0003C;101::AID-YEA66&#x0003E;3.0.CO;2-H" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B54" id="B54"></a>Guti&#x000E9;rrez, A. R., Santamaria, P., Epifano, S., Garijo, P., and Lopez. R. (1999). Ecology of spontaneous fermentation in one winery during consecutive years. <i>Lett. Appl. Microbiol. </i>29, 411&#x02013;415. doi: 10.1046/j.1472-765X.1999.00657.x</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1046/j.1472-765X.1999.00657.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B55" id="B55"></a>Hennequin, C., Thierry, A., Richard, G. F., Lecointre, G., Nguyen, H. V., Gaillardin, C., et al. (2001). Microsatellite typing as a new tool for identification of <i>Saccharomyces cerevisiae</i> strains. <i>J. Clin. Microbiol. </i>39, 551&#x02013;559. doi: 10.1128/JCM.39.2.551-559.2001</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=11158105" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=11158105" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1128/JCM.39.2.551-559.2001" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B56" id="B56"></a>Hyma, K. E., Saerens, S. M., Verstrepen, K. J., and Fay, J. C. (2011). Divergence in wine characteristics produced by wild and domesticated strains of <i>Saccharomyces cerevisiae</i>. <i>FEMS Yeast Res. </i>11, 540&#x02013;551. doi: 10.1111/j.1567-1364.2011.00746.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22093681" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22093681" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2011.00746.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B57" id="B57"></a>Infante, J. J., Dombeck, K., Rebordinos, L., Cantoral, J. M., and Young, E. T. (2003). Genome-wide amplifications caused by chromosomal rearrangements play a major role in the adaptive evolution of natural yeast. <i>Genetics </i>165, 1745&#x02013;1759. doi: 10.1101/gr.436602.</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=14704163" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=14704163" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1101/gr.436602." target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B58" id="B58"></a>Ivey, M. L., and Phister, T. G. (2011). Detection and identification of microorganisms in wine: a review of molecular techniques. <i>J. Ind. Microbiol. Biotechnol. </i>38, 1619&#x02013;1634. doi: 10.1007/s10295-011-1020-x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21822905" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21822905" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s10295-011-1020-x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B59" id="B59"></a>Johnston, J. R., and Mortimer, R. K. (1986). Electrophoretic karyotyping of laboratory and commercial strains of <i>Saccharomyces</i> and other yeasts. <i>Int. J. Syst. Bacteriol. </i>36, 569&#x02013;572. doi: 10.1099/00207713-36-4-569</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1099/00207713-36-4-569" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B60" id="B60"></a>Kurtzman, C. P., Fell, J. W., and Boekhout, T. (2011). &#x0201C;Saccharomyces Meyen ex Reess (1870),&#x0201D; in <i>The Yeasts a Taxonomic Study</i>, 5th Edn, eds C. P. Kurtzman, J. W. Fell, and T. Boekhout (Amsterdam: Elsevier Science), 733&#x02013;746.</p>
</div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B61" id="B61"></a>Kvitek, D. J., Will, J. L., and Gasch, A. P. (2008). Variation in stress sensitivity and genomic expression in diverse <i>S. cerevisiae</i> isolates. <i>PLoS Genet. </i>4:e1000223. doi: 10.1371/journal.pgen.1000223</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18927628" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18927628" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pgen.1000223" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B62" id="B62"></a>Landry, C. R., Towsend, J. P., Hartl, D. L., and Cavalieri, D. (2006). Ecological and evolutionary genomics of <i>Saccharomyces cerevisiae</i>. <i>Mol. Ecol. </i>15, 575&#x02013;591. doi: 10.1111/j.1365-294X.2006.02778.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16499686" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16499686" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-294X.2006.02778.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B63" id="B63"></a>Legras, J. L., and Karst, F. (2003). Optimisation of interdelta analysis for <i>Saccharomyces cerevisiae</i> strain characterization. <i>FEMS Microbiol. Lett. </i>221, 249&#x02013;255. doi: 10.1016/S0378-1097(03)00205-2</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=12725935" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=12725935" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/S0378-1097(03)00205-2" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B64" id="B64"></a>Legras, J. L., Merdinoglu, D., Cornuet, J. M., and Karst, F. (2007). Bread, beer and wine: <i>Saccharomyces cerevisiae</i> diversity reflects human history. <i>Mol. Ecol. </i>16, 2091&#x02013;2102. doi: 10.1111/j.1365-294X.2007.03266.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17498234" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17498234" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-294X.2007.03266.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B65" id="B65"></a>Legras, J. L., Ruh, O., Merdinoglu, D., and Karst, F. (2005). Selection of hypervariable microsatellite loci for the characterization of <i>Saccharomyces cerevisiae</i>. <i>Int. J. Food Microbiol. </i>102, 73&#x02013;83. doi: 10.1016/j.ijfoodmicro.2004.12.007</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15925004" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15925004" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2004.12.007" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B66" id="B66"></a>Le Jeune, C., Erny, C., Demuyter, C., and Lollier, M. (2006). Evolution of the population of <i>Saccharomyces cerevisiae</i> from grape to wine in a spontaneous fermentation. <i>Food Microbiol. </i>23, 709&#x02013;716. doi: 10.1016/j.fm.2006.02.007</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16943073" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16943073" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.fm.2006.02.007" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B67" id="B67"></a>Lelandais, G., Goudot, C., and Devaux, F. (2011). The evolution of gene expression regulatory networks in yeasts. <i>C. R. Biol. </i>334, 655&#x02013;661. doi: 10.1016/j.crvi.2011.05.014</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21819947" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21819947" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.crvi.2011.05.014" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B68" id="B68"></a>Li, S. S., Cheng, C., Li, Z., Chen, J. Y., Yan, B., Han, B. Z., et al. (2010). Yeast species associated with wine grapes in China. <i>Int. J. Food Microbiol. </i>138, 85&#x02013;90. doi: 10.1016/j.ijfoodmicro.2010.01.009</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=20116124" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=20116124" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2010.01.009" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B69" id="B69"></a>Libkind, D., Hittinger, C. T., Val&#x000E9;rio, E., Gon&#x000E9;alves, C., Dover, J., Johnston, M., et al. (2011). Microbe domestication and the identification of the wild genetic stock of lager-brewing yeast. <i>Proc. Natl. Acad. Sci. U.S.A. </i>108, 1539&#x02013;14544. doi: 10.1073/pnas.1105430108</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21873232" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21873232" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1073/pnas.1105430108" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B70" id="B70"></a>Lidzbarsky, G. A., Shkolnik, T., and Nevo, E. (2009). Adaptive response to DNA-damaging agents in natural <i>Saccharomyces cerevisiae</i> populations from &#x0201C;Evolution Canyon&#x0201D;, Mt. Carmel, Israel<i>. PLoS ONE </i>4:e5914. doi: 10.1371/journal.pone.0005914</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19526052" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19526052" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pone.0005914" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B71" id="B71"></a>Liti, G., Carter, D. M., Moses, A. M., Warringer, J., Parts, L., James, S. A., et al. (2009). Population genomics of domestic and wild yeasts. <i>Nature </i>458, 337&#x02013;341. doi: 10.1038/nature07743</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19212322" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19212322" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/nature07743" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B72" id="B72"></a>Liti, G., and Louis, E. J. (2005). Yeast evolution and comparative genomics. <i>Annu. Rev. Microbiol. </i>59, 135&#x02013;153. doi: 10.1146/annurev.micro.59.030804.121400</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15877535" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15877535" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1146/annurev.micro.59.030804.121400" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B73" id="B73"></a>Lopandic, K., Gangl, H., Wallner, E., Tscheik, G., Leitner, G., Querol, A., et al. (2007). Genetically different wine yeasts isolated from Austrian vine-growing regions influence wine aroma differently and contain putative hybrids between <i>Saccharomyces cerevisiae</i> and <i>Saccharomyces kudriavzevii</i>. <i>FEMS Yeast Res. </i>7, 953&#x02013;965. doi: 10.1111/j.1567-1364.2007.00240.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17484739" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17484739" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2007.00240.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B74" id="B74"></a>Lopes, C. A., van Broock, M., Querol, A., and Caballero, A. C. (2002). <i>Saccharomyces cerevisiae</i> wine yeast populations in a cold region in Argentinean Patagonia. A study at different fermentation scales<i>. J. Appl. Microbiol. </i>93, 608&#x02013;615. doi: 10.1046/j.1365-2672.2002.01738.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=12234344" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=12234344" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1046/j.1365-2672.2002.01738.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B75" id="B75"></a>Lopez, V., Fernandez-Espinar, M. T., Barrio, E., Ramon, D., and Querol, A. (2003). A new PCR-based method for monitoring inoculated wine fermentations. <i>Int. J. Food Microbiol. </i>81, 63&#x02013;71. doi: 10.1016/S0168-1605(02)00194-0</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=12423919" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=12423919" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/S0168-1605(02)00194-0" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B76" id="B76"></a>Martinez, C., Cosgaya, P., Vasquez, C., Gac, S., and Ganga, A. (2007). High degree of correlation between molecular polymorphism and geographic origin of wine yeast strains. <i>J. Appl. Microbiol. </i>103, 2185&#x02013;2195. doi: 10.1111/j.1365-2672.2007.03493.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18045401" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18045401" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-2672.2007.03493.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B77" id="B77"></a>Martini, A., and Vaughan-Martini, A. (1990). &#x0201C;Grape must fermentation: past and present,&#x0201D; in <i>Yeast Technology</i>, eds J. F. T. Spencer and D. M. Spencer (Berlin: Springer-Verlag), 105&#x02013;123.</p>
</div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B78" id="B78"></a>Martiny, J. B. H., Bohannan, B. J. M., Brown, J. H., Colwell, R. K., Furhman, J. A., Green, J. L., et al. (2006). Microbial biogeography: putting microorganisms on the map. <i>Nature </i>4, 102&#x02013;112. doi: 10.1038/nrmicro1341</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch= 16415926" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id= 16415926" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/nrmicro1341" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B79" id="B79"></a>McCullough, M. J., Clemons, K. V., McCusker, J. H., and Stevens, D. A. (1998). Intergenic transcribed spacer PCR ribotyping for differentiation of <i>Saccharomyces</i> species and interspecific hybrids. <i>J. Clin. Microbiol. </i>36, 1035&#x02013;1038.</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=9542932" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=9542932" target="_blank">Pubmed Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B80" id="B80"></a>McGovern, P. E., Zhang, J., Tang, J., Zhang, Z., Hall, G. R., Moreau, R. A., et al. (2004). Fermented beverages of pre- and proto-historic China. <i>Proc. Nat. Acad. Sci. U.S.A. </i>101, 17593&#x02013;17598. doi: 10.1073/pnas.0407921102</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15590771" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15590771" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1073/pnas.0407921102" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B81" id="B81"></a>Mesa, J. J., Infante, J. J., Rebordinos, L., and Cantoral, J. M. (1999). Characterization of yeasts involved in the biological ageing of sherry wines. <i>Lebensm. Wiss. Technol. </i>32, 114&#x02013;120. doi: 10.1006/fstl.1998.0514</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1006/fstl.1998.0514" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B82" id="B82"></a>Mortimer, R., and Polsinelli, M. (1999). On the origins of wine yeast. <i>Res. Microbiol. </i>150, 199&#x02013;204. doi: 10.1016/S0923-2508(99)80036-9</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=10229949" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=10229949" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/S0923-2508(99)80036-9" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B83" id="B83"></a>Mortimer, R., Romano, P., Suzzi, G., and Polsinelli, M. (1994). Genome renewal: a new phenomenon revealed from a genetic study of 43 strains of <i>Saccharomyces cerevisiae</i> from natural fermentation of grape musts. <i>Yeast </i>10, 1543&#x02013;1552. doi: 10.1002/yea.320101203</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=7725789" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=7725789" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1002/yea.320101203" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B84" id="B84"></a>Muller, L. A. H., and McCusker, J. H. (2009a). A multi-species based taxonomic microarray reveals interspecies hybridization and introgression in <i>Saccharomyces cerevisiae</i>. <i>FEMS Yeast Res. </i>9, 143&#x02013;152. doi: 10.1111/j.1567-1364.2008.00464.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19054123" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19054123" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2008.00464.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B85" id="B85"></a>Muller, L. A. H., and McCusker, J. H. (2009b). Microsatellite analysis of genetic diversity among clinical and nonclinical <i>Saccharomyces cerevisiae</i> isolates suggests heterozygote advantage in clinical environments. <i>Mol. Ecol. </i>18, 2779&#x02013;2786. doi: 10.1111/j.1365-294X.2009.04234.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19457175" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19457175" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-294X.2009.04234.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B86" id="B86"></a>Muller, L. A. H., and McCusker, J. H. (2011). Nature and distribution of large sequence polymorphisms in <i>Saccharomyces cerevisiae</i>. <i>FEMS Yeast Res. </i>11, 587&#x02013;594. doi: 10.1111/j.1567-1364.2011.00748.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22093685" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22093685" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2011.00748.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B87" id="B87"></a>Nadal, D., Carro, D., Fern&#x000E1;ndez, L. J., and Pina, B. (1999). Analysis and dynamics of the chromosomal complement of wild sparkling-wine yeast strains. <i>Appl. Environ. Microbiol. </i>65, 1688&#x02013;1695.</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=10103269" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=10103269" target="_blank">Pubmed Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B88" id="B88"></a>Naumov, G. I., James, S. A., Naumova, E. S., Louis, E. J., and Roberts, I. N. (2000). Three new species in the <i>Saccharomyces sensu stricto</i> complex: <i>Saccharomyces cariocanus, Saccharomyces kudriavzevii and Saccharomyces mikatae</i>. <i>Int. J. Syst. Evol.Microbiol. </i>50, 1931&#x02013;1942. doi: 10.1099/00207713-50-5-1931</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=11034507" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=11034507" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1099/00207713-50-5-1931" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B89" id="B89"></a>Naumov, G. I., Naumova, E. S., Antunovics, Z., and Sipiczki, M. (2002). <i>Saccharomyces bayanus </i>var. <i>uvarum </i>in Tokaj wine-making of Slovakia and Hungary<i>. Appl. Microbiol.</i> Biotechnol. 59, 727&#x02013;730. doi: 10.1007/s00253-002-1077-6</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=12226732" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=12226732" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s00253-002-1077-6" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B90" id="B90"></a>Naumov, G. I., Serpova, E. V., and Naumova, E. S. (2006). A genetically isolated population of <i>Saccharomyces cerevisiae</i> in Malaysia. <i>Mikrobiologiia </i>75, 245&#x02013;249. doi: 10.1134/S0026261706020147</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16758873" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16758873" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1134/S0026261706020147" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B91" id="B91"></a>Ness, F., Lavallee, F., Dubourdieu, D., Aigle, M., and Dulau, L. (1993). Identification of yeast strains using the polymerase chain reaction. <i>J. Sci. Food Agric. </i>62, 89&#x02013;94. doi: 10.1002/jsfa.2740620113</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1002/jsfa.2740620113" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B92" id="B92"></a>Nisiotou, A. A., and Nychas, G. J. E. (2007). Yeast populations residing on healthy or botrytis-infected grapes from a vineyard in Attica, Greece. <i>Appl. Environ. Microbiol. </i>73, 2765&#x02013;2768. doi: 10.1128/AEM.01864-06</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17293525" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17293525" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1128/AEM.01864-06" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B93" id="B93"></a>Novo, M., Bigey, F., Beyne, E., Galeote, V., Gavory, F., Mallet, S., et al. (2009). Eucaryote-to-eucaryote gene transfer events revealed by the genome sequence of the wine yeast <i>Saccharomyces cerevisiae</i> EC1118. <i>Proc. Nat. Acad. Sci. U.S.A. </i>106, 16333&#x02013;16338. doi: 10.1073/pnas.0904673106</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19805302" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19805302" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1073/pnas.0904673106" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B94" id="B94"></a>O&#x02019;Malley, M. (2008). Everything is everywhere: but the environment selects&#x02019;: ubiquitous distribution and ecological determinism in microbial biogeography. <i>Stud. Hist. Phil. Biol. Biomed. Sci.</i> 39, 314&#x02013;325. doi: 10.1016/j.shpsc.2008.06.005.</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18761283" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18761283" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.shpsc.2008.06.005" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B95" id="B95"></a>Perez-Ortin, J. E., Querol, A., Puig, S., and Barrio, E. (2002). Molecular characterization of a chromosomal rearrangement involved in the adaptive evolution of wine yeast strains. <i>Genome Res. </i>12, 1533&#x02013;1539. doi: 10.1101/gr.436602</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=12368245" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=12368245" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1101/gr.436602" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B96" id="B96"></a>Povhe, J. K., Cadez, N., Zagorc, T., Bubic, V., Zupec, A., and Raspor, P. (2001). Yeast population dynamics in five spontaneous fermentations of Malvasia must. <i>Food Microbiol. </i>18, 247&#x02013;259. doi: 10.1006/fmicb.2001.0396</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1006/fmic.2001.0396" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B97" id="B97"></a>Prakitchaiwattana, C. J., Fleet, G. H., and Heard, G. M. (2004). Application and evaluation of denaturing gradient gel electrophoresis to analyse the yeast ecology of wine grapes. <i>FEMS Yeast Res. </i>4, 865&#x02013;877. doi: 10.1016/j.femsyr.2004.05.004</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15450194" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15450194" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.femsyr.2004.05.004" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B98" id="B98"></a>Querol, A., and Bond, U. (2009). The complex and dynamic genomes of industrial yeasts. <i>FEMS Yeast Res. </i>293, 1&#x02013;10. doi: 10.1111/j.1574-6968.2008.01480.x. </p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19175410" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19175410" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1574-6968.2008.01480.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B99" id="B99"></a>Querol, A., Barrio, E., and Ramon, D. (1994). Population dynamics of natural <i>Saccharomyces</i> strains during wine fermentation. <i>Int. J. Food Microbiol. </i>21, 315&#x02013;323. doi: 10.1016/0168-1605(94)90061-2</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=8043350" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=8043350" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/0168-1605(94)90061-2" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B100" id="B100"></a>Quesada, M. P., and Cenis, J. L. (1995). Use of random amplified polymorphic DNA (RAPD-PCR) in the characterization of wine yeasts. <i>Am. J. Enol. Vitic. </i>46, 204&#x02013;208. doi: 10.1099/00207713-49-1-329</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1099/00207713-49-1-329" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B101" id="B101"></a>Ramette, A., and Tiedje, J. M. (2007). Biogeography: an emerging cornerstone for understanding prokaryotic diversity, ecology, and evolution. <i>Microb. Ecol. </i>53, 197&#x02013;207. doi: 10.1007/s00248-005-5010-2</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17106806" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17106806" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s00248-005-5010-2" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B102" id="B102"></a>Raspor, P., Milek, D. M., Polanc, J., Mozina, S. S., and Cadez, N. (2006). Yeasts isolated from three varieties of grapes cultivated in different locations of the Dolenjska vine-growing region, Slovenia. <i>Int. J. Food Microbiol. </i>109, 97&#x02013;102. doi: 10.1016/j.ijfoodmicro.2006.01.017 PMCid:3662647</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16626833" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16626833" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2006.01.017" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B103" id="B103"></a>Replansky, T., Koufopanou, V., Greig, D., and Bell, G. (2008). <i>Saccharomyces sensu stricto</i> as a model for evolution and ecology. <i>Trends Ecol. Evol. </i>23, 494&#x02013;501. doi: 10.1016/j.tree.2008.05.005</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18656281" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18656281" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.tree.2008.05.005" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B104" id="B104"></a>Richards, K. D., Goddard, M. R., and Gardner, C. (2009). A database of microsatellite genotypes for <i>Saccharomyces cerevisiae</i>. <i>Antonie van Leeuwenhoek </i>96, 355&#x02013;359. doi: 10.1007/s10482-009-9346-3</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19396625" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19396625" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s10482-009-9346-3" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B105" id="B105"></a>Rolland, T., Dujon, B., and Richard, G. F. (2010). Dynamic evolution of megasatellites in yeasts. <i>Nucleic Acids Res. </i>38, 4731&#x02013;4739. doi: 10.1093/nar/gkq207</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=20360043" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=20360043" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1093/nar/gkq207" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B106" id="B106"></a>Rolland, T., Neuv&#x000E9;glise, C., Sacerdot, C., and Dujon, B. (2009). Insertion of horizontally transferred genes within conserved syntenic regions of yeast genomes. <i>PLoS ONE </i>4: e6515.
 doi: 10.1371/journal.pone.0006515</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19654869" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19654869" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pone.0006515" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B107" id="B107"></a>Sabate, J., Cano, J., Querol, A., and Guillamon, J. M. (1998). Diversity of <i>Saccharomyces </i>strains in wine fermentations: analysis for two consecutive years. <i>Lett. Appl. Microbiol. </i>26, 452&#x02013;455. doi: 10.1046/j.1472-765X.1998.00369.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=9717318" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=9717318" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1046/j.1472-765X.1998.00369.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B108" id="B108"></a>Sampaio, J. P., and Goncalves, P. (2008). Natural populations of <i>Saccharomyces kudriavzevii</i> in Portugal are associated with oak bark and are sympatric with <i>S.</i> <i>cerevisiae </i>and<i> S. paradoxus. Appl. Environ. Microbiol. </i>74, 2144&#x02013;2152. doi: 10.1128/AEM.02396-07</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=18281431" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=18281431" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1128/AEM.02396-07" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B109" id="B109"></a>Scannell, D. R., Zill, O. A., Rokas, A., Payen, C., Dunham, M. J., Eisen, M. B., et al. (2011). The awesome power of yeast evolutionary genetics: new genome sequences and strain resources for the <i>Saccharomyces sensu stricto</i> genus. <i>Investigation </i>1, 11&#x02013;25. doi: 10.1534/g3.111.000273. </p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22384314" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22384314" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1534/g3.111.000273." target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B110" id="B110"></a>Schacherer, J., Shapiro, J. A., Ruderfer, D. M., and Kruglyak, L. (2009). Comprehensive polymorphism survey elucidates population structure of <i>Saccharomyces cerevisiae</i>. <i>Nature </i>458, 342&#x02013;345. doi: 10.1038/nature07670</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19212320" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19212320" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/nature07670" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B111" id="B111"></a>Schuller, D., Alves, H., Dequin, S., and Casal, M. (2005). Ecological survey of <i>Saccharomyces cerevisiae</i> strains from vineyards in the Vinho Verde Region of Portugal. <i>FEMS Microbiol. Ecol. </i>51, 167&#x02013;177. doi: 10.1016/j.femsec.2004.08.003</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=16329865" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=16329865" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.femsec.2004.08.003" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B112" id="B112"></a>Schuller, D., Cardoso, F., Sousa, S., Gomes, P., Gomes, A. C., Santos, M. A. S., et al. (2012). Genetic diversity and population structure of <i>Saccharomyces cerevisiae</i> strains isolated from different grape varieties and winemaking regions. <i>PLoS ONE </i>7: e32507. doi: 10.1371/journal.pone.0032507</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22393409" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22393409" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pone.0032507" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B113" id="B113"></a>Schuller, D., and Casal, M. (2007). The genetic structure of fermentative vineyard-associated <i>Saccharomyces cerevisiae</i> populations revealed by microsatellite analysis. <i>Antonie van Leeuwenhoek </i>91, 137&#x02013;150. doi: 10.1007/s10482-006-9104-8</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17094015" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17094015" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s10482-006-9104-8" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B114" id="B114"></a>Schuller, D., Pereira, L., Alves, H., Cambon, B., Dequin, S., and Casal, M. (2007). Genetic characterization of commercial <i>Saccharomyces cerevisiae</i> isolates recovered from vineyard environments. <i>Yeast </i>24,625&#x02013;636. doi: 10.1002/yea.1496</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17534867" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17534867" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1002/yea.1496" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B115" id="B115"></a>Schuller, D., Valero, E., Dequin, S., and Casal, M. (2004). Survey of molecular methods for the typing of wine yeast strains. <i>FEMS Microbiol. Lett. </i>231, 19&#x02013;26. doi: 10.1016/S0378-1097(03)00928-5</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=14769461" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=14769461" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/S0378-1097(03)00928-5" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B116" id="B116"></a>Schwartz, D. C., and Cantor, C. R. (1984). Separation of yeast chromosome sized DNAs by pulsed field gradient gel electrophoresis. <i>Cell </i>37, 65&#x02013;67. doi: 10.1016/0092-8674(84)90301-5</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=6373014" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=6373014" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/0092-8674(84)90301-5" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B117" id="B117"></a>Sicard, D., and Legras, J. L. (2011). Bread, beer and wine: yeast domestication in the <i>Saccharomyces sensu stricto</i> complex. <i>C. R. Biol. </i>334, 229&#x02013;236. doi: 10.1016/j.crvi.2010.12.016</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21377618" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21377618" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.crvi.2010.12.016" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B118" id="B118"></a>Sipiczki, M. (2011). Diversity, variability and fast adaptive evolution of the wine yeast (<i>Saccharomyces cerevisiae</i>) genome&#x02013;-a review. <i>Ann. Microbiol. </i>61, 85&#x02013;93. doi: 10.1007/s13213-010-0086-4</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1007/s13213-010-0086-4" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B119" id="B119"></a>Sipiczki, M., Romano, P., Capece, A., and Paraggio, M. (2004). Genetic segregation of natural <i>Saccharomyces cerevisiae</i> strains derived from spontaneous fermentation of Aglianico wine. <i>J. Appl. Microbiol. </i>96, 1169&#x02013;1175. doi: 10.1111/j.1365-2672.2004.02254.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15078535" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15078535" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-2672.2004.02254.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B120" id="B120"></a>Sipiczki, M., Romano, P., Lipani, G., Miklos, I., and Antunovics, Z. (2001). Analysis of yeasts derived from natural fermentation in a Tokaj winery. <i>Antonie van Leeuwenhoek </i>79, 97&#x02013;105. doi: 10.1023/A:1010249408975</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=11392490" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=11392490" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1023/A:1010249408975" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B121" id="B121"></a>Stefanini, I., Dapporto, L., Legras, J. L., Calabretta, A., Di Paola, M., De Filippo, C., et al. (2012). Role of social wasps in <i>Saccharomyces cerevisiae</i> ecology and evolution. <i>Proc. Natl. Acad. Sci. U.S.A. </i>109, 13398&#x02013;13403. doi: 10.1073/pnas.1208362109</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22847440" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22847440" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1073/pnas.1208362109" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B122" id="B122"></a>Steinmetz, L. M., Sinha, H., Richards, D. R., Spiegelman, J. I., Oefner, P. J., McCusker, J. H., et al. (2002). Dissecting the architecture of a quantitative trait locus in yeast. <i>Nature </i>416, 326&#x02013;330. doi: 10.1038/416326a</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=11907579" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=11907579" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/416326a" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B123" id="B123"></a>Suzzi, G. (2011). From wild strain to domesticated strain: the philosophy of microbial diversity in foods. <i>Front. Microbiol. </i>2:169. doi: 10.3389/fmicb.2011.00169</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21887153" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21887153" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.3389/fmicb.2011.00169" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B124" id="B124"></a>Suzzi, G., Arfelli, G., Schirone, M., Corsetti, A., Perpetuini, G., and Tofalo, R. (2012). Effect of grape indigenous <i>Saccharomyces cerevisiae</i> strains on Montepulciano d&#x02019;Abruzzo red wine quality. <i>Food Res. Int. </i>46, 22&#x02013;29. doi: 10.1016/j.foodres.2011.10.046</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1016/j.foodres.2011.10.046" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B125" id="B125"></a>Takahashi, T. (1978). Genetic analysis of a German wine yeast. <i>Bull. Brew. Sci. </i>24, 39&#x02013;47. doi: 10.1007/BF00429312</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1007/BF00429312" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B126" id="B126"></a>Thornton, R. J. (1986). Genetic characterization of New Zealand and Australian wine yeasts. <i>Antonie van Leeuwenhoek </i>52, 97&#x02013;103. doi: 10.1007/BF00429312</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=3729379" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=3729379" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/BF00429312" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B127" id="B127"></a>Tofalo, R., Torriani, S., Chaves-L&#x000F3;pez, C., Martuscelli, M., Paparella, A., and Suzzi, G. (2007). Asurvey of <i>Saccharomyces</i> populations associated with wine fermentations from the Apulia region (South Italy). <i>Ann. Microbiol. </i>57, 545&#x02013;552. doi: 10.1007/BF03175353</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1007/BF03175353" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B128" id="B128"></a>Valero, E., Cambon, B., Schuller, D., Casal, M., and Dequin, S. (2007). Biodiversity of <i>Saccharomyces</i> yeast strains from grape berries of wine-producing areas using starter commercial yeasts. <i>FEMS Yeast Res. </i>7, 317&#x02013;329. doi: 10.1111/j.1567-1364.2006.00161.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17040482" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17040482" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1567-1364.2006.00161.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B129" id="B129"></a>Valero, E., Schuller, D., Cambon, B., Casal, M., and Dequin, S. (2005). Dissemination and survival of commercial wine yeast in the vineyard: a large-scale, three-years study. <i>FEMS Yeast Res. </i>5, 959&#x02013;969. doi: 10.1016/j.femsyr.2005.04.007</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=15975857" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=15975857" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.femsyr.2005.04.007" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B130" id="B130"></a>Versavaud, A., Couroux, P., Roulland, C., Dulau, C., and Hallet, J. N. (1995). Genetic diversity and geographical distribution of wild <i>Saccharomyces cerevisiae</i> strains from wine-producing area of Charentes, France. <i>Appl. Environ. Microbiol. </i>61, 3521&#x02013;3529. doi: 10.1016/S0168-1605(00)00287-7</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=7486988" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=7486988" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/S0168-1605(00)00287-7" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B131" id="B131"></a>Vezinhet, F., Blondin, B., and Hallet, J. N. (1990). Chromosomal DNA pattern and mitochondrial DNA polymorphism as tool for identification of enological strains of <i>Saccharomyces cerevisiae</i>. <i>Appl. Microbiol. Biotechnol. </i>32, 568&#x02013;571. doi: 10.1007/BF00173729</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1007/BF00173729" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B132" id="B132"></a>Vezinhet, F., Hallet, J. N., Valade, M., and Poulard, A. (1992). Ecological survey of wine yeast strains by molecular methods of identification. <i>Am. J. Enol. Vitic. </i>43, 83&#x02013;86. doi: 10.1111/j.1365-2672.2006.03150.x</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1111/j.1365-2672.2006.03150.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B133" id="B133"></a>Vigentini, I., Fracassetti, D., Picozzi, C., and Foschino, R. (2009). Polymorphisms of <i>Saccharomyces cerevisiae</i> genes involved in wine production. <i>Curr. Microbiol. </i>58, 211&#x02013;218. doi: 10.1007/s00284-008-9310-x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=19005725" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=19005725" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1007/s00284-008-9310-x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B134" id="B134"></a>Wang, Q. M., Liu, W. Q., Liti, G., Wang, S. A., and Bai, F. Y. (2012). Surprisingly diverged populations of <i>Saccharomyces cerevisiae</i> in natural environments remote from human activity. <i>Mol. Ecol. </i>21, 5404&#x02013;5417. doi: 10.1111/j.1365-294X.2012.05732.x</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=22913817" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=22913817" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-294X.2012.05732.x" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B135" id="B135"></a>Warringer, J., Zorgo, E., Cubillos, F. A., Zia, A., Gjuvsland, A., Simpson, J. D., et al. (2011). Trait variation in yeast is defined by population history. <i>PLoS Genet. </i>7:e1002111. doi: 10.1371/journal.pgen.1002111</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=21698134" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=21698134" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1371/journal.pgen.1002111" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B136" id="B136"></a>Wei, W., McCusker, J. H., Hyman, R. W., Jones, T., Ning, Ye., Cao, Z., et al. (2007). Genome sequencing and comparative analysis of <i>Saccharomyces cerevisiae</i> strain. <i>Proc. Natl. Acad. Sci. U.S.A. </i>104, 12825&#x02013;12830. doi: 10.1073/pnas.0701291104</p>
<p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&#x00026;Cmd=ShowDetailView&#x00026;TermToSearch=17652520" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&#x00026;cmd=prlinks&#x00026;retmode=ref&#x00026;id=17652520" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1073/pnas.0701291104" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B137" id="B137"></a>Weiher, E., and Keddy, P. A. (1995). The assembly of experimental wetland plant communities. <i>Oikos</i> 73, 323&#x02013;335. doi: 10.2307/3545956</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.2307/3545956" target="_blank">CrossRef Full Text</a></p></div>
<div class="References" style="margin-bottom:0.5em;">
<p class="ReferencesCopy1"><a name="B138" id="B138"></a>Weiss, M.M., Hermsen, M.A.J.A., Meijer, G.A., van Grieken, N.C.T., Baak, J.P.A., Kuipers, E.J., et al. (1999). Comparative genomic hybridization. <i>J. Clin. Pathol. Mol. Pathol. </i>52, 243&#x02013;251. doi: 10.1136/mp.52.5.243</p>
<p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1136/mp.52.5.243" target="_blank">CrossRef Full Text</a></p>
</div>
</div>
<div class="thinLineM20"></div>
<div class="AbstractSummary">
<p><span>Keywords</span>: <i>Saccharomyces cerevisiae</i>, wine, biogeography, molecular methods, <i>terroir</i></p>
<p><span>Citation:</span> Tofalo R, Perpetuini G, Schirone M, Fasoli G, Aguzzi I, Corsetti A and Suzzi G (2013) Biogeographical characterisation of Saccharomyces cerevisiae wine yeast by molecular methods. <i>Front. Microbiol.</i> <b>4</b>:166. <a href="http://dx.doi.org/10.3389/fmicb.2013.00166">doi: 10.3389/fmicb.2013.00166</a></p>
<p id="timestamps">
<span>Received:</span> 13 February 2013; <span>Accepted:</span> 03 June 2013; <br/><span>Published online:</span> 24 June 2013.</p>
<div>
<p>Edited by:</p>
<a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=DaniloErcolini&#x00026;UID=18040">Danilo Ercolini</a>, Universit&#x000E0; degli Studi di Napoli Federico II, Italy
</div>
<div>
<p>Reviewed by:</p>
<a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=CatarinaPrista&#x00026;UID=23269">Catarina Prista</a>, Instituto Superior de Agronomia, Portugal<br/>
<a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=ChrysoulaTassou&#x00026;UID=35945">Chrysoula C. Tassou</a>, National Agricultural Research Foundation, Greece
</div>
<p><span>Copyright:</span> &#x000A9; 2013 Tofalo, Perpetuini, Schirone, Fasoli, Aguzzi, Corsetti and Suzzi. This is an open-access article distributed under the terms of the <a id="lnkNonCommercialLicense" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution License</a>, which permits use, distribution and reproduction in other forums, provided the original authors and source are credited and subject to any copyright notices concerning any third-party graphics etc.</p>
<p><span>*Correspondence:</span> Rosanna Tofalo and Giovanna Suzzi, Faculty of BioScience and Technology for Food, Agriculture and Environment, University of Teramo, Via C.R. Lerici 1, 64023 Mosciano Sant&#x02019;Angelo, TE, Italy e-mail: rtofalo@unite.it; gsuzzi@unite.it</p>
<div class="clear"></div>
</div>
<html xmlns:mso="urn:schemas-microsoft-com:office:office" xmlns:msdt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882"><head>
<!--[if gte mso 9]><xml>
<mso:CustomDocumentProperties>
<mso:StageName msdt:dt="string"></mso:StageName>
<mso:DocumentType msdt:dt="string">Finalhtml</mso:DocumentType>
<mso:TitleName msdt:dt="string">fmicb-04-00166-r2.html</mso:TitleName>
<mso:FileFormat msdt:dt="string">html</mso:FileFormat>
<mso:DocumentId msdt:dt="string">fmicb-04-00166-r2.html</mso:DocumentId>
</mso:CustomDocumentProperties>
</xml><![endif]-->
</head>                    <div class="thin-line-dark"></div>
                    <div class="social-feed">
                    </div>
                </div>
                <div class="span4">
                    <div class="right-container">
    <div class="downloads-content">
        <ul>
                    <li>
                        <a href="javascript:void(0);" class="popover-trigger popover-enhanced " data-type="pdf">
                            <i class="fr-icon-small dropdown-pdf"></i>
                            <span>PDF</span>
                        </a>
                            <input type="hidden" data-name="Download PDF" data-url="/Journal/10.3389/fmicb.2013.00166/pdf" />
                            <input type="hidden" data-name="View Enhanced PDF" data-url="http://www.readcube.com/articles/10.3389/fmicb.2013.00166" />
                    </li>

                <li>
                    <a href="javascript:void(0);" class="popover-trigger" data-type="xml">
                        <i class="fr-icon-small xml"></i>
                        <span>XML</span>
                    </a>
                            <input type="hidden" data-name="NLM" data-url="/Journal/10.3389/fmicb.2013.00166/xml/nlm" data-webtrendtracking="&#39;{&quot;WT_action&quot;:&quot;xml_download&quot;,&quot;WT_destination&quot;:&quot;xml_nlm&quot;,&quot;WT_source&quot;:&quot;article&quot;,&quot;WT_dl&quot;:&quot;51&quot;,&quot;WT_ndl&quot;:&quot;51&quot;}&#39;" />
                            <input type="hidden" data-name="JATS" data-url="/Journal/10.3389/fmicb.2013.00166/xml/jats" data-webtrendtracking="&#39;{&quot;WT_action&quot;:&quot;xml_download&quot;,&quot;WT_destination&quot;:&quot;xml_jats&quot;,&quot;WT_source&quot;:&quot;article&quot;,&quot;WT_dl&quot;:&quot;51&quot;,&quot;WT_ndl&quot;:&quot;51&quot;}&#39;" />
                </li>
                            <li>
                    <a href="javascript:void(0);" class="popover-trigger" data-type="citation">
                        <i class="fr-icon-mini citation"></i>
                        <span class="citation-wrap">Export Citation</span>
                    </a>
                        <input type="hidden" data-name="EndNote" data-url="http://www.frontiersin.org/Journal/DownloadFile.ashx?cit=1&amp;aid=46996&amp;cid=21&amp;fe=.enw" data-webtrendtracking="&#39;{&quot;WT_action&quot;:&quot;citation_download&quot;,&quot;WT_destination&quot;:&quot;citation_endnote&quot;,&quot;WT_source&quot;:&quot;article&quot;,&quot;WT_dl&quot;:&quot;51&quot;,&quot;WT_ndl&quot;:&quot;51&quot;}&#39;" />
                        <input type="hidden" data-name="Reference Manager" data-url="http://www.frontiersin.org/Journal/DownloadFile.ashx?cit=1&amp;aid=46996&amp;cid=21&amp;fe=.ris" data-webtrendtracking="&#39;{&quot;WT_action&quot;:&quot;citation_download&quot;,&quot;WT_destination&quot;:&quot;citation_reference_manager&quot;,&quot;WT_source&quot;:&quot;article&quot;,&quot;WT_dl&quot;:&quot;51&quot;,&quot;WT_ndl&quot;:&quot;51&quot;}&#39;" />
                        <input type="hidden" data-name="Simple TEXT file" data-url="http://www.frontiersin.org/Journal/DownloadFile.ashx?cit=1&amp;aid=46996&amp;cid=21&amp;fe=.txt" data-webtrendtracking="&#39;{&quot;WT_action&quot;:&quot;citation_download&quot;,&quot;WT_destination&quot;:&quot;citation_simple_text_file&quot;,&quot;WT_source&quot;:&quot;article&quot;,&quot;WT_dl&quot;:&quot;51&quot;,&quot;WT_ndl&quot;:&quot;51&quot;}&#39;" />
                        <input type="hidden" data-name="BibTex" data-url="http://www.frontiersin.org/Journal/DownloadFile.ashx?cit=1&amp;aid=46996&amp;cid=21&amp;fe=.bib" data-webtrendtracking="&#39;{&quot;WT_action&quot;:&quot;citation_download&quot;,&quot;WT_destination&quot;:&quot;citation_bibtex&quot;,&quot;WT_source&quot;:&quot;article&quot;,&quot;WT_dl&quot;:&quot;51&quot;,&quot;WT_ndl&quot;:&quot;51&quot;}&#39;" />
                </li>
        </ul>
    </div>
    <div class="article-share-count">
        <div class="total-views" title="The total view count is updated once a day, so not to worry if you don&#39;t see immediate results.">
            <div>
                <span class="count"></span>
                <span class="name">total views</span>

            </div>
        </div>
            <div class="altmetric-icon">
                <div class='altmetric-embed' data-badge-type='1' data-doi='10.3389/fmicb.2013.00166' data-link-target="new"></div>
            </div>
                    <div class="buttons">

                <a class="btn btn-impact" href="http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/impact#impact" data-webtrendtracking='{ "WT_action": "article_impact_click", "WT_destination": "article_impact_page", "WT_source": "article", "WT_dl": "1", "WT_ndl": "1"  }'><i class="fr-icon-mini impact"></i>View Article Impact</a>

            </div>
        <div class="thin-line"></div>
        <h4 class="main-heading">FRONTIERS</h4>
        <div class="frontiers-network" style="display: none">
            <div class="button-likes">
                <button class="btn btn-white like" type="button" data-webtrendtracking='{ "WT_action": "like", "WT_destination": "like", "WT_source": "article", "WT_dl": "51", "WT_ndl": "51"  }'>Like</button>
                <span class="final-counts"></span>
            </div>
            <div class="button-comments">
                <button class="btn btn-white comment" type="button">Comment</button>
                <span class="final-counts"></span>
            </div>
            <div class="button-shares">
                <button class="btn btn-white share" type="button">Share</button>
                <span class="final-counts"></span>
            </div>
        </div>
        <div class="thin-line"></div>
        <h4 class="main-heading">SHARE ON</h4>

        <div class="share-media">

            <div class="social_block ">
                <div class="atButton">
                    <a class='addthis_button_facebook' addthis:title="Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods" addthis:description="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the divers..." addthis:url="http://www.frontiersin.org/Journal/10.3389/fmicb.2013.00166"></a>
                </div>
                <a><span class="facebook_count"></span></a>
            </div>


            <div class="social_block">
                <div class="atButton">
                    <a class='addthis_button_twitter' addthis:title="Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods" addthis:description="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the divers..." addthis:url="http://www.frontiersin.org/Journal/10.3389/fmicb.2013.00166"></a>
                </div>
                <a><span class="twitter_count"></span></a>
            </div>


            <div class="social_block">
                <div class="atButton">
                    <a class='addthis_button_google' addthis:title="Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods" addthis:description="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the divers..." addthis:url="http://www.frontiersin.org/Journal/10.3389/fmicb.2013.00166"></a>
                </div>
                <a><span class="googleplus_count"></span></a>

            </div>

            <div class="social_block">
                <div class="atButton">
                    <a class='addthis_button_linkedin' addthis:title="Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods" addthis:description="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the divers..." addthis:url="http://www.frontiersin.org/Journal/10.3389/fmicb.2013.00166"></a>
                </div>
                <a><span class="linkedin_count"></span></a>

            </div>

            <div class="social_block">
                <div class="atButton">
                    <a class='addthis_button_more' addthis:title="Biogeographical characterization of Saccharomyces cerevisiae wine yeast by molecular methods" addthis:description="Biogeography is the descriptive and explanatory study of spatial patterns and processes involved in the distribution of biodiversity. Without biogeography, it would be difficult to study the divers..." href="#" addthis:url="http://www.frontiersin.org/Journal/10.3389/fmicb.2013.00166"></a>
                </div>
                <a><span class="total_count"></span></a>
            </div>


        </div>

    </div>

        <div class="widget-listing">
            <h3>TABLE OF CONTENTS</h3>

<ul class="flyoutJournal">
<li><a href="#h1">Abstract</a></li>
<li><a href="#h2">Introduction</a></li>
<li><a href="#h3"><i>Saccharomyces cerevisiae</i> Wine Yeast</a></li>
<li><a href="#h4">Molecular Methods</a></li>
<li><a href="#h5">Conclusion</a></li>
<li><a href="#h6">Conflict of Interest Statement</a></li>
<li><a href="#h7">Acknowledgments</a></li>
<li><a href="#h8">References</a></li>
</ul>
<html xmlns:mso="urn:schemas-microsoft-com:office:office" xmlns:msdt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882"><head>
<!--[if gte mso 9]><xml>
<mso:CustomDocumentProperties>
<mso:StageName msdt:dt="string"></mso:StageName>
<mso:DocumentType msdt:dt="string">Finalhtml</mso:DocumentType>
<mso:TitleName msdt:dt="string">fmicb-04-00166-r2-menu.html</mso:TitleName>
<mso:FileFormat msdt:dt="string">html</mso:FileFormat>
<mso:DocumentId msdt:dt="string">fmicb-04-00166-r2-menu.html</mso:DocumentId>
</mso:CustomDocumentProperties>
</xml><![endif]-->
</head>        </div>


    <div class="widget-listing commentary-article hide">
    </div>

    <div class="widget-listing original-article hide">
    </div>

        <div class="widget-listing people-also-looked-at hide">
        </div>
</div>

                </div>
            </div>
        </div>
    </div>
</div>




    </div>
   
</div>
<div class="modal-container impact-modal-container dp-none">

</div>
<div id="divTemplates">
    



<script type="text/template" id="template-impact-modal">
    <div class="impact-modal" id="impact-modal" tabindex="-1">
        <div class="container">
            <div class="row">
                <div style="padding-bottom: 100px">
                    <div class="modal-dialog">
                        <div class="modal-content">
                            <div class="modal-header clearfix">
                                <button type="button" class="close btn-close" data-dismiss="modal" aria-hidden="true">&times;</button>
                                <h3 class="modal-title" id="myModalLabel">Impact</h3>
                                <h6><a href="#" data-dismiss="modal" class="article-title"></a></h6>
                            </div>
                            <div class="modal-body">
                                <img class="loader" src="/Images/Frontiers/Common/Icon/loading.gif" alt="Loading.." />

                                <div class="row article-impact-tabs hide">
                                    <!-- Nav tabs -->
                                    <ul class="nav nav-tabs impact-tabs clearfix">
                                        <li class="active"><a href="#views-tab" data-toggle="tab" data-tab-type="views"><span class="total-views" data-tab-type="views"></span>Views</a></li>
                                        <li><a href="#demographics-tab" data-toggle="tab" data-tab-type="demographics">Demographics</a></li>
                                        <li><a href="#social-buzz-tab" data-toggle="tab" data-tab-type="socialbuzz">Social Buzz</a></li>
                                    </ul>
                                    <div class="tab-content impact-tabs">
                                        <!-- VIEWS TAB -->
                                        <div class="tab-pane fade in active" id="views-tab">
                                            <h5 style="padding-left: 68px;" class="sincebeginning">Since beginning</h5>
                                            <h5 style="padding-left: 68px; display: none;" class="thisyear">This year</h5>
                                            <h5 style="padding-left: 68px; display: none;" class="thismonth">This month</h5>


                                            <p style="display: none;height:60px" class="views-empty-message lead">No records found</p>
                                            <div class="chart-container">
                                                <p class="info sincebeginning"><strong class="begin-views-downloads download-font-styles"> </strong> views and downloads <strong class="begin-views year-font-styles"> </strong> views <strong class="begin-downloads year-font-styles"> </strong> downloads</p>
                                                <p class="info thisyear" style="display: none;"><strong class="year-views-downloads download-font-styles"> </strong> views and downloads <strong class="year-views year-font-styles"> </strong> views <strong class="year-downloads year-font-styles"> </strong> downloads</p>
                                                <p class="info thismonth" style="display: none;"><strong class="month-views-downloads download-font-styles"> </strong> views and downloads <strong class="month-views year-font-styles"> </strong> views <strong class="month-downloads year-font-styles"> </strong> downloads</p>
                                                <div id="views-timeline-chart" class="views-bar-charts"></div>
                                                <div id="views-bar-chart" class="views-timeline-charts"></div>
                                            </div>
                                            <div id="views-chart-controls" class="input-block-level">
                                                <!-- Split button -->
                                                <div class="btn-group pull-right">
                                                    <button type="button" class="btn btn-flat dropdown-toggle" data-toggle="dropdown">
                                                        <span id="Period" data-bind="label">Select a time period</span>&nbsp;<span class="caret"></span>
                                                    </button>
                                                    <ul id="PeriodDrop" class="dropdown-menu" role="menu">
                                                        <li class="beginning"><a>Since beginning</a></li>
                                                        <li class="year"><a>This year</a></li>
                                                        <li class="month"><a>This month</a></li>
                                                    </ul>
                                                </div>
                                                <div style="padding-left: 72px;" class="btn-group pull-left" data-toggle="tooltip" title="Toggle between timeline and bar chart views">
                                                    <button type="button" class="btn btn-flat btn-line active" id="views-timeline-button"><span style="padding: 0 4px">&nbsp;&nbsp;</span></button>
                                                    <button type="button" class="btn btn-flat btn-line" id="views-bar-button"><span style="padding: 0 4px">&nbsp;&nbsp;</span></button>

                                                </div>
                                                <div class="clearfix"></div>
                                            </div><!-- /#views-chart-controls -->

                                        </div><!-- /#views-tab -->
                                        <!-- DEMOGRAPHICS TAB -->
                                        <div class="tab-pane fade" id="demographics-tab">

                                            <div class="btn-group pull-right" id="demographics-dropdown" style="margin-right: 10px;">
                                                &nbsp;
                                            </div>

                                            <div id="map-container">

                                                <div id="map">

                                                </div>
                                                <img class="map-loading" src="/Images/Frontiers/Common/Icon/loading.gif" alt="Loading.." />
                                            </div>
                                            <div class="clearfix"></div>

                                            <div class="row">
                                                <div class="col-xs-6">
                                                    <h5>Top countries</h5>
                                                    <div id="demographics-top-countries"></div>
                                                </div>
                                                <div class="col-xs-6">
                                                    <h5>Top referring sites</h5>
                                                    <div id="demographics-top-sites"></div>
                                                </div>
                                            </div>
                                            <hr>
                                            <div class="row">
                                                <div class="col-xs-4">
                                                    <h5 style="text-align: center;">Domain</h5>
                                                    <div id="demographics-domain"></div>
                                                </div>
                                                <div class="col-xs-4">
                                                    <h5 style="text-align: center;">Field</h5>
                                                    <div id="demographics-field"></div>
                                                </div>
                                                <div class="col-xs-4">
                                                    <h5 style="text-align: center;">Specialty</h5>
                                                    <div id="demographics-specialty"></div>
                                                </div>
                                            </div>
                                            <hr>
                                            <div class="row">
                                                <div class="col-xs-4">
                                                    <h5 style="text-align: center;">Industry</h5>
                                                    <div id="demographics-industry"></div>
                                                </div>

                                                <div class="col-xs-4">
                                                    <h5 style="text-align: center;">Education</h5>
                                                    <div id="demographics-education"></div>
                                                </div>

                                                <div class="col-xs-4">
                                                    <h5 style="text-align: center;">Position</h5>
                                                    <div id="demographics-position"></div>
                                                </div>
                                            </div>
                                            <div class="clearfix"></div>

                                            <div class="demographics-age-gender">
                                                <hr>
                                                <h5 style="margin: 35px 0px;">Age and Gender</h5>
                                                <div id="demographics-age" style="min-width: 810px; height: 400px; margin: 0 auto"></div>
                                            </div>
                                        </div><!-- /#demographics-tab -->
                                        <!-- SOCIAL BUZZ TAB -->
                                        <div class="tab-pane fade" id="social-buzz-tab">
                                            <h5 class="altmetric-title">Altmetric</h5>
                                            <div id="altmetric-donut" data-badge-details="right" data-badge-type="medium-donut" data-doi="<%- doi %>" data-hide-no-mentions="false" class="altmetric-embed"></div>

                                            <div id='main' class="social-buzz-content-area">
                                                <div id='content'>
                                                    <div id='details'>
                                                        <div class="row">
                                                            <div class="col-xs-6">
                                                                <h5>Sources</h5>
                                                            </div>
                                                            <div class="col-xs-6">
                                                                <ul class="nav nav-tabs pull-right">
                                                                    <li class='active'><a href="#altmetric-all-tab" data-toggle="tab">All</a></li>
                                                                    <li id='tab_for_msm'><a href="#details_msm" data-toggle="tab">News</a></li>
                                                                    <li id='tab_for_blogs'><a href="#details_blogs" data-toggle="tab">Blogs</a></li>
                                                                </ul>
                                                            </div>
                                                        </div>

                                                        <div class="tabbable">
                                                            <div class="tab-content">

                                                                <div class='details tab-pane ' id='details_blogs'>

                                                                    <div class="blogs-posts"></div>
                                                                </div> <!--Blog-->

                                                                <div class='details tab-pane' id='details_msm'>

                                                                    <div class="news-posts"></div>
                                                                </div> <!--News-->

                                                                <div class='details tab-pane active' id='altmetric-all-tab'>
                                                                    <div class="all-posts"></div>
                                                                </div> <!--All-->

                                                            </div>
                                                        </div>

                                                    </div>

                                                    <div class='clearfix'></div>
                                                </div>
                                            </div>                                           

                                        </div><!-- /#social-buzz-tab -->
                                        <!-- OVERALL TAB -->
                                        <div class="tab-pane fade" id="overall-tab">
                                            <div id="overall-chart"></div>
                                            <div class="clearfix"></div>
                                        </div><!-- /#overall-tab -->
                                    </div>
                                </div><!-- /.row -->

                            </div><!-- /.modal-body -->
                            <div class="modal-footer">
                                <button type="button" class="btn btn-new-orange btn-flat btn-close" data-dismiss="modal">Close</button>
                            </div>
                        </div><!-- /.modal-content -->
                    </div><!-- /.modal-dialog -->
                </div><!-- /.col-xs-9 -->
            </div><!-- /.row -->
        </div><!-- /.container -->
    </div>
</script>



<script type="text/template" id="template-social-buzz-blogs-posts">
    <div class='posts_info'>So far Altmetric has seen <b><%- blogsCount %></b> posts.</div>
    <% _.each( socialBuzzPosts, function( socialBuzzPost ){ %>

    <div class='postbox'>
        <div class='title'><a target='_blank' href="<%- socialBuzzPost.Url%>"><%- socialBuzzPost.Title %></a></div>
        <div class='source'><a target='_blank' href="<%- socialBuzzPost.Author.Url %>"><%- socialBuzzPost.Author.Name %></a></div>
        <div class='snippet'>
            <p><%- socialBuzzPost.Summary %></p>
        </div>
        <div class="footer blog">
            <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn) %>
        </div>
    </div>

    <% }); %>

</script>


<script type="text/template" id="template-social-buzz-news-posts">
    <div class='posts_info'>So far Altmetric has seen <b><%- newsCount %></b> stories.</div>
    <% _.each( socialBuzzPosts, function( socialBuzzPost ){ %>
    <div class='postbox'>
        <div class='author source' style='width: 80px;'>
            <div style='max-width: 80px; overflow: hidden;'>
                <img width='80' src="<%- socialBuzzPost.Author.Image %>" class="photo" />
            </div>
        </div>

        <div class="entry-content" style="margin-left: 0px;">
            <div class='title'>
                <a target='_blank' href="<%- socialBuzzPost.Url%>" ><%- socialBuzzPost.Title %> </a>
            </div>
            <div class='source'><%- socialBuzzPost.Author.Name %></div>
            <div class='snippet'>
                <p><%- socialBuzzPost.Summary %></p>
            </div>
        </div>
        <div class="footer">
           <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn) %>
        </div>
    </div>

    <% }); %>

</script>


<script type="text/template" id="template-social-buzz-all-posts">

    <div class="posts_info">So far Altmetric has seen <b><%- allCount %></b> stories.</div>

    <% _.each( socialBuzzPosts, function( socialBuzzPost ){ %>
    <% if (socialBuzzPost.Type == "Blogs") {%>

    <div class='postbox'>
        <div class='title'><a target='_blank' href="<%- socialBuzzPost.Url%>"><%- socialBuzzPost.Title %></a></div>
        <div class='source'><a target='_blank' href="<%- socialBuzzPost.Author.Url %>"><%- socialBuzzPost.Author.Name %></a></div>
        <div class='snippet'>
            <p><%- socialBuzzPost.Summary %></p>
        </div>
        <div class="footer blog">            
            <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn) %>
        </div>
    </div>

    <% } %>
    <%  if(socialBuzzPost.Type == "News"){ %>
    <div class='postbox'>
        <div class='author source' style='width: 80px;'>
            <div style='max-width: 80px; overflow: hidden;'>
                <img width='80' src="<%- socialBuzzPost.Author.Image %>" class="photo" />
            </div>
        </div>

        <div class="entry-content" style="margin-left: 0px;">
            <div class='title'>
                <a target='_blank' href="<%- socialBuzzPost.Url%>"><%- socialBuzzPost.Title %> </a>
            </div>
            <div class='source'><%- socialBuzzPost.Author.Name %></div>
            <div class='snippet'>
                <p><%- socialBuzzPost.Summary %></p>
            </div>
        </div>
        <div class="footer">
            <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn)%>
        </div>
    </div>
    <% } %>
    <% }); %>

</script>
</div>
<div class="modal-backdrop fade in hide"></div>


        
<div class="container">
    <div class="backTop"><a href="#">Back to top</a></div>
</div>
<div class="footer">
    <div class="container">
        <div class="row">
            <div class="nav">
                <ul>
                    <li><a href="http://www.frontiersin.org/Journal/Frontiers.aspx">Home</a></li>
                    <li><a href="http://www.frontiersin.org/about">About Frontiers</a></li>
                    <li><a href="http://www.frontiersin.org/about/journalseries">Journals A-Z</a></li>
                    <li><a href="http://www.frontiersin.org/about/Institutional_Membership">Institutional Membership</a></li>

                </ul>
                <ul>
                    <li><a href="http://www.frontiersin.org/about/contact">Contact</a></li>
                    <li><a href="http://www.frontiersin.org/press">Press Relations</a></li>
                    <li><a href="http://www.frontiersin.org/news/all_news">News</a></li>
                </ul>
                <ul>
                    <li><a href="http://www.frontiersin.org/submissioninfo">Submit</a></li>
                    <li><a href="http://www.frontiersin.org/about/faq">FAQs</a></li>
                    <li><a href="http://www.frontiersin.org/TermsandConditions.aspx">Terms &amp; Conditions</a></li>
                </ul>
                <ul>
                    <li><a href="http://www.frontiersin.org/journal/alerts.aspx">Newsletters</a></li>
                    <li><a href="http://www.frontiersin.org/blog/Frontiers_Social_Media_and_RSS/496">RSS/Twitter</a></li>
                    <li><a href="http://www.frontiersin.org/Team.aspx">Team</a></li>
                    <li><a href="http://www.frontiersin.org/Careers">Careers</a></li>

                </ul>
                <span class="ftrCols">
                    <img src="/Images/Frontiers/Common/Footer/footer.gif" alt="logo" />
                </span>
            </div>
        </div>
    </div>
    <div id="copyright">
        <p>© 2007 - 2014 Frontiers Media S.A. All Rights Reserved</p>
    </div>
</div>






        <script src="/bundles/frontiers-basic-js?v=HsJuu6zxwIxY3YXoHgfZBQ03Rr2yWzdcf_FyhIoOdf81"></script>

        <script src="https://api-journal.frontiersin.org/Configuration/Bootstrapper?url=http://journal.frontiersin.org/Journal/10.3389/fmicb.2013.00166/full"></script>

        <script src="https://api-journal.frontiersin.org/iBar/Bootstrapper"></script>

        <script src="/bundles/frontiers-common-js?v=IqnAlvMqkdGSPqbM_Vm72oELqTrAc9m8Q5AXXDSJEXI1"></script>


        
    <script type="text/javascript">
    var FRArticle = (function() {
        return {
            ArticleId: '46996',
            DOI:'10.3389/fmicb.2013.00166',
            LoginUserId: '0',
            DomainId: '1',
            FieldId: '51',
            SpecialityId: '441',
            IsPreview: FRSafe.boolean('False')
        };
    })();

    var FRSocial = (function() {
        return {
            itemId: 14,
            itemTypeId: 1,
            entityId: '46996',
            ownerId: '45789',
            sanPath: 'http://www.frontiersin.org/files/',
            subItemId: '8',
            loginUserId: '0',
            ownerNWDBId: '10',
            pageType :1

        };
    })();
    </script>

    
    <script type="text/javascript">

        if (!FRArticle.IsPreview) {

            (function(i, s, o, g, r, a, m) {
                i['GoogleAnalyticsObject'] = r;
                i[r] = i[r] || function() {
                    (i[r].q = i[r].q || []).push(arguments)
                }, i[r].l = 1 * new Date();
                a = s.createElement(o),
                m = s.getElementsByTagName(o)[0];
                a.async = 1;
                a.src = g;
                m.parentNode.insertBefore(a, m)
            })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');

            ga('create', 'UA-9164039-1', 'frontiersin.org');
            ga('send', 'pageview');

        }
    </script>

    
   <script src="/bundles/frontiers-journal-js?v=p2RDXuQPGcsN4-o3Rkbacfba-wORLZ2r0nAj3gCnBU41"></script>



    </body>
</html>
